target/linux/mediatek/dts/mt7986b-tplink-re6000xd.dts

```diff
@@ -0,0 +1,296 @@
+		};
+		lan1 {
+			gpios = <&pio 16 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_LAN;
+			function-enumerator = <0>;
+		};
+		lan2 {
+			gpios = <&pio 10 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_LAN;
+			function-enumerator = <1>;
+		};
+		lan3 {
+			gpios = <&pio 18 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_LAN;
+			function-enumerator = <2>;
+		};
+	};
+};
+
+&crypto {
+	status = "okay";
+};
+
+&eth {
+	status = "okay";
+
+	gmac0: mac@0 {
+		compatible = "mediatek,eth-mac";
+		reg = <0>;
+		phy-mode = "2500base-x";
+
+		fixed-link {
+			speed = <2500>;
+			full-duplex;
+			pause;
+		};
+	};
+
+	gmac1: mac@1 {
+		compatible = "mediatek,eth-mac";
+		reg = <1>;
+		phy-handle = <&phy6>;
+		phy-mode = "2500base-x";
+	};
+
+	mdio: mdio-bus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+	};
+};
+
+&mdio {
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	reset-gpios = <&pio 6 GPIO_ACTIVE_LOW>;
+	reset-delay-us = <1500000>;
+	reset-post-delay-us = <1000000>;
+
+	/* LAN3 2.5Gbps phy
+	   MaxLinear GPY211C0VC (SLNW8) */
+	phy6: phy@6 {
+		compatible = "ethernet-phy-ieee802.3-c45";
+		reg = <6>;
+	};
+
+	switch: switch@1f {
+		compatible = "mediatek,mt7531";
+		reg = <31>;
+		reset-gpios = <&pio 5 GPIO_ACTIVE_HIGH>;
+	};
+};
+
+&switch {
+	ports {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		/* reorder LAN1 and LAN2 port to match the port order of the case
+			LAN1 - LAN2 - LAN3 (top to bottom of the case, no silkscreen)
+		*/
+		/* LAN2 port */
+		port@1 {
+			reg = <1>;
+			label = "lan2";
+		};
+
+		/* LAN1 port */
+		port@2 {
+			reg = <2>;
+			label = "lan1";
+		};
+
+		port@6 {
+			reg = <6>;
+			ethernet = <&gmac0>;
+			phy-mode = "2500base-x";
+
+			fixed-link {
+				speed = <2500>;
+				full-duplex;
+				pause;
+			};
+		};
+	};
+};
+
+&pio {
+	spi_flash_pins: spi-flash-pins-33-to-38 {
+		mux {
+			function = "spi";
+			groups = "spi0", "spi0_wp_hold";
+		};
+		conf-pu {
+			pins = "SPI2_CS", "SPI2_HOLD", "SPI2_WP";
+			drive-strength = <8>;
+			mediatek,pull-up-adv = <0>; /* bias-disable */
+		};
+		conf-pd {
+			pins = "SPI2_CLK", "SPI2_MOSI", "SPI2_MISO";
+			drive-strength = <8>;
+			mediatek,pull-down-adv = <0>; /* bias-disable */
+		};
+	};
+
+	wf_2g_5g_pins: wf_2g_5g-pins {
+		mux {
+			function = "wifi";
+			groups = "wf_2g", "wf_5g";
+		};
+		conf {
+			pins = "WF0_HB1", "WF0_HB2", "WF0_HB3", "WF0_HB4",
+			       "WF0_HB0", "WF0_HB0_B", "WF0_HB5", "WF0_HB6",
+			       "WF0_HB7", "WF0_HB8", "WF0_HB9", "WF0_HB10",
+			       "WF0_TOP_CLK", "WF0_TOP_DATA", "WF1_HB1",
+			       "WF1_HB2", "WF1_HB3", "WF1_HB4", "WF1_HB0",
+			       "WF1_HB5", "WF1_HB6", "WF1_HB7", "WF1_HB8",
+			       "WF1_TOP_CLK", "WF1_TOP_DATA";
+			drive-strength = <4>;
+		};
+	};
+};
+
+&spi0 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&spi_flash_pins>;
+	status = "okay";
+
+	spi_nand_flash: flash@0 {
+		compatible = "spi-nand";
```

Comment:
Is this device using `mediatek,nmbm`? 
Could you please post the boot log of the vendor bootloader and vendor firmware.

Reply:
Imho it would be worth going for full U-Boot replacement and dropping nmbm as well as the 2nd UBI partition. Installation anyway requires serial access, so if something goes wrong you could always use `mtk_uartboot`.

Reply:
AFAIK it's not using mediatek.nmbm

This is the spi_nand node from the stock dts:

```
spi_nand@1 {
            #address-cells = <0x1>;
            #size-cells = <0x1>;
            compatible = "spi-nand";
            reg = <0x1>;
            spi-max-frequency = <0x1312D00>;
            spi-tx-buswidth = <0x4>;
            spi-rx-buswidth = <0x4>;
            partition@0 {
                label = "boot";
                reg = <0x0 0x200000>;
                read-only;
            };
            partition@200000 {
                label = "u-boot-env";
                reg = <0x200000 0x100000>;
            };
            partition@300000 {
                label = "ubi0";
                reg = <0x300000 0x3200000>;
            };
            partition@3500000 {
                label = "ubi1";
                reg = <0x3500000 0x3200000>;
            };
            partition@6700000 {
                label = "userconfig";
                reg = <0x6700000 0x800000>;
            };
            partition@6f00000 {
                label = "tp_data";
                reg = <0x6F00000 0x400000>;
                phandle = <0xF>;
            };
        };
    };
```

And the stock bootlog:

```
F0: 102B 0000

FA: 1040 0000

FA: 1040 0000 [0200]

F9: 0000 0000

V0: 0000 0000 [0001]

00: 0000 0000

BP: 2400 0041 [0000]

G0: 1190 0000

EC: 0000 0000 [1000]

T0: 0000 0141 [010F]

Jump to BL


NOTICE:  BL2: v2.6(release):b8c8172fd-dirty
NOTICE:  BL2: Built : 15:49:17, Sep 23 2022
NOTICE:  WDT: disabled
NOTICE:  EMI: Using DDR3 settings
NOTICE:  EMI: Detected DRAM size: 512MB
NOTICE:  EMI: complex R/W mem test passed
NOTICE:  SPI_NAND parses attributes from parameter page.
NOTICE:  SPI_NAND Detected ID 0x35
NOTICE:  Page size 2048, Block size 131072, size 134217728
NOTICE:  BL2: Booting BL31
NOTICE:  BL31: v2.6(release):b8c8172fd-dirty
NOTICE:  BL31: Built : 15:49:21, Sep 23 2022


U-Boot 2022.01-rc4 (Sep 23 2022 - 15:48:25 +0800)

CPU:   MediaTek MT7986
Model: mt7986-rfb
DRAM:  512 MiB
MMC:   mmc@11230000: 0
Loading Environment from MTD... spi-nand: spi_nand spi_nand@1: GigaDevice SPI NAND was found.
spi-nand: spi_nand spi_nand@1: 128 MiB, block size: 128 KiB, page size: 2048, OOB size: 64
*** Warning - bad CRC, using default environment

In:    serial@11002000
Out:   serial@11002000
Err:   serial@11002000
Net:   
Warning: ethernet@15100000 (eth0) using random MAC address - d2:4b:b7:16:41:1f
eth0: ethernet@15100000
press ctrl-c or t to go to uboot cmdline
 0 new bootarg:ubi.mtd=ubi0 console=ttyS0,115200n1 loglevel=8 earlycon=uart8250,mmio32,0x11002000 init=/etc/preinit
ubi0: attaching mtd3
ubi0: scanning is finished
ubi0: attached mtd3 (name "ubi0", size 50 MiB)
ubi0: PEB size: 131072 bytes (128 KiB), LEB size: 126976 bytes
ubi0: min./max. I/O unit sizes: 2048/2048, sub-page size 2048
ubi0: VID header offset: 2048 (aligned 2048), data offset: 4096
ubi0: good PEBs: 400, bad PEBs: 0, corrupted PEBs: 0
ubi0: user volume: 3, internal volumes: 1, max. volumes count: 128
ubi0: max/mean erase counter: 1/0, WL threshold: 4096, image sequence number: 664357216
ubi0: available PEBs: 268, total reserved PEBs: 132, PEBs reserved for bad PEB handling: 20
Reading from volume 'uboot' to 0x41dfffc0, size 0x0 ... OK
Reading from volume 'rootfs' to 0x4a000000, size 0x0 ... OK
Reading from volume 'kernel' to 0x46000000, size 0x0 ... OK
## Booting kernel from Legacy Image at 41dfffc0 ...
   Image Name:   seconduboot
   Image Type:   ARM Linux Standalone Program (uncompressed)
   Data Size:    746832 Bytes = 729.3 KiB
   Load Address: 00000000
   Entry Point:  41e00000
   Verifying Checksum ... OK
   Loading Standalone Program


U-Boot 2022.01-rc4 (Sep 23 2022 - 15:47:39 +0800)

CPU:   MediaTek MT7986
Model: mt7986-rfb
DRAM:  512 MiB
MMC:   mmc@11230000: 0
Loading Environment from MTD... spi-nand: spi_nand spi_nand@1: GigaDevice SPI NAND was found.
spi-nand: spi_nand spi_nand@1: 128 MiB, block size: 128 KiB, page size: 2048, OOB size: 64
*** Warning - bad CRC, using default environment

In:    serial@11002000
Out:   serial@11002000
Err:   serial@11002000
Net:   
Warning: ethernet@15100000 (eth0) using random MAC address - 3e:37:f7:04:b8:d8
eth0: ethernet@15100000
new bootarg:ubi.mtd=ubi0 console=ttyS0,115200n1 loglevel=8 earlycon=uart8250,mmio32,0x11002000 init=/etc/preinit
second uboot to boot kernel @46000000
## Loading kernel from FIT Image at 46000000 ...
   Using 'config-1' configuration
   Trying 'kernel-1' kernel subimage
     Description:  ARM64 OpenWrt Linux-5.4.124
     Type:         Kernel Image
     Compression:  lzma compressed
     Data Start:   0x460000e8
     Data Size:    3335100 Bytes = 3.2 MiB
     Architecture: AArch64
     OS:           Linux
     Load Address: 0x48080000
     Entry Point:  0x48080000
     Hash algo:    crc32
     Hash value:   322112ee
     Hash algo:    sha1
     Hash value:   b41b267c8b681a2e64845b08f4386135d18236f0
   Verifying Hash Integrity ... crc32+ sha1+ OK
## Loading fdt from FIT Image at 46000000 ...
   Using 'config-1' configuration
   Trying 'fdt-1' fdt subimage
     Description:  ARM64 OpenWrt mt7986b-ax6000-snand-rfb device tree blob
     Type:         Flat Device Tree
     Compression:  uncompressed
     Data Start:   0x4632e5ec
     Data Size:    17698 Bytes = 17.3 KiB
     Architecture: AArch64
     Hash algo:    crc32
     Hash value:   d6926e94
     Hash algo:    sha1
     Hash value:   065224428e52832998ae9b2b813a229c11528506
   Verifying Hash Integrity ... crc32+ sha1+ OK
   Booting using the fdt blob at 0x4632e5ec
   Uncompressing Kernel Image
   Loading Device Tree to 000000005f7f3000, end 000000005f7fa521 ... OK

Starting kernel ...

[    0.000000] Booting Linux on physical CPU 0x0000000000 [0x410fd034]
[    0.000000] Linux version 5.4.124 (soho@ubuntu) (gcc version 8.4.0 (OpenWrt GCC 8.4.0 r0-35e974e68)) #0 SMP Fri Sep 2 03:28:28 2022
[    0.000000] Machine model: MediaTek MT7986b RFB
[    0.000000] earlycon: uart8250 at MMIO32 0x0000000011002000 (options '')
[    0.000000] printk: bootconsole [uart8250] enabled
[    0.000000] On node 0 totalpages: 130064
[    0.000000]   DMA32 zone: 2048 pages used for memmap
[    0.000000]   DMA32 zone: 0 pages reserved
[    0.000000]   DMA32 zone: 130064 pages, LIFO batch:31
[    0.000000] psci: probing for conduit method from DT.
[    0.000000] psci: PSCIv1.1 detected in firmware.
[    0.000000] psci: Using standard PSCI v0.2 function IDs
[    0.000000] psci: MIGRATE_INFO_TYPE not supported.
[    0.000000] psci: SMC Calling Convention v1.0
[    0.000000] percpu: Embedded 20 pages/cpu s44248 r8192 d29480 u81920
[    0.000000] pcpu-alloc: s44248 r8192 d29480 u81920 alloc=20*4096
[    0.000000] pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
[    0.000000] Detected VIPT I-cache on CPU0
[    0.000000] CPU features: detected: ARM erratum 845719
[    0.000000] CPU features: detected: GIC system register CPU interface
[    0.000000] CPU features: kernel page table isolation disabled by kernel configuration
[    0.000000] Built 1 zonelists, mobility grouping on.  Total pages: 128016
[    0.000000] Kernel command line: ubi.mtd=ubi0 console=ttyS0,115200n1 loglevel=8 earlycon=uart8250,mmio32,0x11002000 init=/etc/preinit
[    0.000000] Dentry cache hash table entries: 65536 (order: 7, 524288 bytes, linear)
[    0.000000] Inode-cache hash table entries: 32768 (order: 6, 262144 bytes, linear)
[    0.000000] mem auto-init: stack:off, heap alloc:off, heap free:off
[    0.000000] Memory: 491676K/520256K available (6718K kernel code, 514K rwdata, 1928K rodata, 448K init, 290K bss, 28580K reserved, 0K cma-reserved)
[    0.000000] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
[    0.000000] rcu: Hierarchical RCU implementation.
[    0.000000] rcu: 	CONFIG_RCU_FANOUT set to non-default value of 32.
[    0.000000] rcu: RCU calculated value of scheduler-enlistment delay is 10 jiffies.
[    0.000000] NR_IRQS: 64, nr_irqs: 64, preallocated irqs: 0
[    0.000000] GICv3: GIC: Using split EOI/Deactivate mode
[    0.000000] GICv3: 640 SPIs implemented
[    0.000000] GICv3: 0 Extended SPIs implemented
[    0.000000] GICv3: Distributor has no Range Selector support
[    0.000000] GICv3: 16 PPIs implemented
[    0.000000] GICv3: no VLPI support, no direct LPI support
[    0.000000] GICv3: CPU0: found redistributor 0 region 0:0x000000000c080000
[    0.000000] random: get_random_bytes called from start_kernel+0x2ec/0x438 with crng_init=0
[    0.000000] arch_timer: cp15 timer(s) running at 13.00MHz (phys).
[    0.000000] clocksource: arch_sys_counter: mask: 0xffffffffffffff max_cycles: 0x2ff89eacb, max_idle_ns: 440795202429 ns
[    0.000002] sched_clock: 56 bits at 13MHz, resolution 76ns, wraps every 4398046511101ns
[    0.008163] Calibrating delay loop (skipped), value calculated using timer frequency.. 26.00 BogoMIPS (lpj=130000)
[    0.018569] pid_max: default: 32768 minimum: 301
[    0.023272] Mount-cache hash table entries: 1024 (order: 1, 8192 bytes, linear)
[    0.030623] Mountpoint-cache hash table entries: 1024 (order: 1, 8192 bytes, linear)
[    0.039168] ASID allocator initialised with 65536 entries
[    0.044645] rcu: Hierarchical SRCU implementation.
[    0.049692] smp: Bringing up secondary CPUs ...
[    0.054459] Detected VIPT I-cache on CPU1
[    0.054476] GICv3: CPU1: found redistributor 1 region 0:0x000000000c0a0000
[    0.054494] CPU1: Booted secondary processor 0x0000000001 [0x410fd034]
[    0.054759] Detected VIPT I-cache on CPU2
[    0.054771] GICv3: CPU2: found redistributor 2 region 0:0x000000000c0c0000
[    0.054780] CPU2: Booted secondary processor 0x0000000002 [0x410fd034]
[    0.054998] Detected VIPT I-cache on CPU3
[    0.055007] GICv3: CPU3: found redistributor 3 region 0:0x000000000c0e0000
[    0.055015] CPU3: Booted secondary processor 0x0000000003 [0x410fd034]
[    0.055047] smp: Brought up 1 node, 4 CPUs
[    0.111602] SMP: Total of 4 processors activated.
[    0.116324] CPU features: detected: 32-bit EL0 Support
[    0.121482] CPU features: detected: CRC32 instructions
[    0.126737] CPU: All CPU(s) started at EL2
[    0.130853] alternatives: patching kernel code
[    0.137287] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
[    0.147189] futex hash table entries: 1024 (order: 4, 65536 bytes, linear)
[    0.154178] pinctrl core: initialized pinctrl subsystem
[    0.159897] NET: Registered protocol family 16
[    0.164546] DMA: preallocated 256 KiB pool for atomic allocations
[    0.171099] cpuidle: using governor ladder
[    0.175265] cpuidle: using governor menu
[    0.187134] cryptd: max_cpu_qlen set to 1000
[    0.192571] SCSI subsystem initialized
[    0.196423] libata version 3.00 loaded.
[    0.200427] usbcore: registered new interface driver usbfs
[    0.205966] usbcore: registered new interface driver hub
[    0.211317] usbcore: registered new device driver usb
[    0.216819] workqueue: max_active 576 requested for napi_workq is out of range, clamping between 1 and 512
[    0.227049] rbus 18000000.wbsys: PCI host bridge to bus 0000:00
[    0.232998] pci_bus 0000:00: root bus resource [mem 0x18000000-0x18ffffff]
[    0.239990] pci_bus 0000:00: root bus resource [bus 00-ff]
[    0.245499] pci_bus 0000:00: scanning bus
[    0.249533] pci 0000:00:00.0: [14c3:7986] type 00 class 0x000280
[    0.255577] pci 0000:00:00.0: reg 0x10: [mem 0x18000000-0x1800000f 64bit]
[    0.262397] pci 0000:00:00.0: reg 0x18: [mem 0x00000000-0x0000000f]
[    0.268695] pci 0000:00:00.0: reg 0x1c: [mem 0x00000000-0x0000000f]
[    0.274989] pci 0000:00:00.0: reg 0x20: [mem 0x00000000-0x0000000f]
[    0.281287] pci 0000:00:00.0: reg 0x24: [mem 0x00000000-0x0000000f]
[    0.288264] pci_bus 0000:00: fixups for bus
[    0.292467] pci_bus 0000:00: bus scan returning with max=00
[    0.298294] clocksource: Switched to clocksource arch_sys_counter
[    0.304950] thermal_sys: Registered thermal governor 'fair_share'
[    0.304952] thermal_sys: Registered thermal governor 'bang_bang'
[    0.311087] thermal_sys: Registered thermal governor 'step_wise'
[    0.317118] thermal_sys: Registered thermal governor 'user_space'
[    0.323153] thermal_sys: Registered thermal governor 'power_allocator'
[    0.329471] NET: Registered protocol family 2
[    0.340469] IP idents hash table entries: 8192 (order: 4, 65536 bytes, linear)
[    0.348045] tcp_listen_portaddr_hash hash table entries: 256 (order: 0, 4096 bytes, linear)
[    0.356459] TCP established hash table entries: 4096 (order: 3, 32768 bytes, linear)
[    0.364264] TCP bind hash table entries: 4096 (order: 4, 65536 bytes, linear)
[    0.371474] TCP: Hash tables configured (established 4096 bind 4096)
[    0.377909] UDP hash table entries: 256 (order: 1, 8192 bytes, linear)
[    0.384478] UDP-Lite hash table entries: 256 (order: 1, 8192 bytes, linear)
[    0.391552] NET: Registered protocol family 1
[    0.395942] PCI: CLS 0 bytes, default 64
[    0.400563] workingset: timestamp_bits=46 max_order=17 bucket_order=0
[    0.409435] squashfs: version 4.0 (2009/01/31) Phillip Lougher
[    0.415298] jffs2: version 2.2 (NAND) (SUMMARY) (LZMA) (RTIME) (CMODE_PRIORITY) (c) 2001-2006 Red Hat, Inc.
[    0.444825] Serial: 8250/16550 driver, 3 ports, IRQ sharing disabled
[    0.451758] printk: console [ttyS0] disabled
[    0.476164] 11002000.serial: ttyS0 at MMIO 0x11002000 (irq = 11, base_baud = 2500000) is a ST16650V2
[    0.485370] printk: console [ttyS0] enabled
[    0.485370] printk: console [ttyS0] enabled
[    0.493722] printk: bootconsole [uart8250] disabled
[    0.493722] printk: bootconsole [uart8250] disabled
[    0.503877] mtk_rng 1020f000.trng: registered RNG driver
[    0.504108] random: fast init done
[    0.509319] cacheinfo: Unable to detect cache hierarchy for CPU 0
[    0.512800] random: crng init done
[    0.520428] loop: module loaded
[    0.525859] mt7986-pinctrl 1001f000.pinctrl: pin_config_set op failed for pin 36
[    0.533264] mtk-spi 1100a000.spi: Error applying setting, reverse things back
[    0.541019] No nand flash found, try esmt
[    0.545037] spi-nand spi0.1: ESMT SPI NAND was found.
[    0.550081] spi-nand spi0.1: 128 MiB, block size: 128 KiB, page size: 2048, OOB size: 64
[    0.558282] 6 fixed-partitions partitions found on MTD device spi0.1
[    0.564621] Creating 6 MTD partitions on "spi0.1":
[    0.569418] 0x000000000000-0x000000200000 : "boot"
[    0.576559] 0x000000200000-0x000000300000 : "u-boot-env"
[    0.583188] 0x000000300000-0x000003500000 : "ubi0"
[    0.639386] 0x000003500000-0x000006700000 : "ubi1"
[    0.695976] 0x000006700000-0x000006f00000 : "userconfig"
[    0.709878] 0x000006f00000-0x000007300000 : "tp_data"
[    0.719752] libphy: Fixed MDIO Bus: probed
[    0.735672] libphy: mdio: probed
[    0.783715] mtk_soc_eth 15100000.ethernet: generated random MAC address 2e:da:47:38:c1:58
[    0.792111] mtk_soc_eth 15100000.ethernet eth0: mediatek frame engine at 0xffffffc011500000, irq 120
[    0.801241] mtk_soc_eth 15100000.ethernet: generated random MAC address 9e:25:b2:ad:a7:82
[    0.809582] mtk_soc_eth 15100000.ethernet eth1: mediatek frame engine at 0xffffffc011500000, irq 120
[    0.818698] (unnamed net_device) (dummy): netif_napi_add() called with weight 256
[    0.826388] xhci-mtk 11200000.xhci: 11200000.xhci supply vbus not found, using dummy regulator
[    0.835022] xhci-mtk 11200000.xhci: 11200000.xhci supply vusb33 not found, using dummy regulator
[    0.843953] mtk-tphy usb-phy@11e10000: Requested PHY is disabled
[    0.850264] xhci-mtk 11200000.xhci: xHCI Host Controller
[    0.855575] xhci-mtk 11200000.xhci: new USB bus registered, assigned bus number 1
[    0.866022] xhci-mtk 11200000.xhci: hcc params 0x01403f99 hci version 0x110 quirks 0x0000000000210010
[    0.875248] xhci-mtk 11200000.xhci: irq 129, io mem 0x11200000
[    0.881454] hub 1-0:1.0: USB hub found
[    0.885208] hub 1-0:1.0: 2 ports detected
[    0.889418] xhci-mtk 11200000.xhci: xHCI Host Controller
[    0.894715] xhci-mtk 11200000.xhci: new USB bus registered, assigned bus number 2
[    0.902187] xhci-mtk 11200000.xhci: Host supports USB 3.2 Enhanced SuperSpeed
[    0.909339] usb usb2: We don't know the algorithms for LPM for this host, disabling LPM.
[    0.917630] hub 2-0:1.0: USB hub found
[    0.921386] hub 2-0:1.0: 1 port detected
[    0.925631] usbcore: registered new interface driver uas
[    0.930977] usbcore: registered new interface driver usb-storage
[    0.937034] i2c /dev entries driver
[    0.941465] mtk-wdt 1001c000.watchdog: Watchdog enabled (timeout=31 sec, nowayout=0)
[    0.949367] device-mapper: ioctl: 4.41.0-ioctl (2019-09-16) initialised: dm-devel@redhat.com
[    5.058360] lp5523x 0-0032: device detection err: -110
[    5.063491] lp5523x: probe of 0-0032 failed with error -110
[    5.069256] crypto-safexcel 10320000.crypto: EIP97:230(0,1,4,4)-HIA:270(0,5,5),PE:150/433,alg:7fcdfc00
[    5.080995] exFAT: Version 1.3.0
[    5.084378] GACT probability on
[    5.087509] Mirror/redirect action on
[    5.091181] u32 classifier
[    5.093873]     input device check on
[    5.097517]     Actions configured
[    5.100999] gre: GRE over IPv4 demultiplexor driver
[    5.105857] ip_gre: GRE over IPv4 tunneling driver
[    5.111285] Initializing XFRM netlink socket
[    5.115814] NET: Registered protocol family 10
[    5.120885] Segment Routing with IPv6
[    5.124563] NET: Registered protocol family 17
[    5.129035] Bridge firewalling registered
[    5.133076] 8021q: 802.1Q VLAN Support v1.8
[    5.353117] libphy: dsa slave smi: probed
[    5.357969] mt7530 mdio-bus:1f lan0 (uninitialized): PHY [dsa-0.0:00] driver [Generic PHY]
[    5.367394] mt7530 mdio-bus:1f lan1 (uninitialized): PHY [dsa-0.0:01] driver [Generic PHY]
[    5.376737] mt7530 mdio-bus:1f lan2 (uninitialized): PHY [dsa-0.0:02] driver [Generic PHY]
[    5.386068] mt7530 mdio-bus:1f lan3 (uninitialized): PHY [dsa-0.0:03] driver [Generic PHY]
[    5.394772] mt7530 mdio-bus:1f: configuring for fixed/2500base-x link mode
[    5.401905] DSA: tree 0 setup
[    5.402157] mt7530 mdio-bus:1f: Link is Up - 2.5Gbps/Full - flow control rx/tx
[    5.405040] ubi0: attaching mtd2
[    5.704481] ubi0: scanning is finished
[    5.716692] ubi0: attached mtd2 (name "ubi0", size 50 MiB)
[    5.722167] ubi0: PEB size: 131072 bytes (128 KiB), LEB size: 126976 bytes
[    5.729023] ubi0: min./max. I/O unit sizes: 2048/2048, sub-page size 2048
[    5.735788] ubi0: VID header offset: 2048 (aligned 2048), data offset: 4096
[    5.742730] ubi0: good PEBs: 400, bad PEBs: 0, corrupted PEBs: 0
[    5.748719] ubi0: user volume: 3, internal volumes: 1, max. volumes count: 128
[    5.755918] ubi0: max/mean erase counter: 1/0, WL threshold: 4096, image sequence number: 664357216
[    5.764940] ubi0: available PEBs: 268, total reserved PEBs: 132, PEBs reserved for bad PEB handling: 20
[    5.774314] ubi0: background thread "ubi_bgt0d" started, PID 742
[    7.458696] block ubiblock0_2: created from ubi0:2(rootfs)
[    7.464165] ubiblock: device ubiblock0_2 (rootfs) set to be root filesystem
[    7.471114] hctosys: unable to open rtc device (rtc0)
[    7.476565] UBIFS (ubi0:2): static UBI volume - read-only mode
[    7.482397] UBIFS error (ubi0:2 pid 1): ubifs_mount: cannot mount read-write - read-only media
[    7.493633] VFS: Mounted root (squashfs filesystem) readonly on device 253:0.
[    7.500936] Freeing unused kernel memory: 448K
[    7.558341] Run /etc/preinit as init process
procd: Console is alive
procd: - preinit -
mounting /dev/root
procd: - early -
procd: - ubus -
procd: - init -
Please press Enter to activate this console.
[    9.589818] ubi1: attaching mtd5
[    9.616065] ubi1: scanning is finished
[    9.628182] ubi1: attached mtd5 (name "tp_data", size 4 MiB)
[    9.633833] ubi1: PEB size: 131072 bytes (128 KiB), LEB size: 126976 bytes
[    9.640730] ubi1: min./max. I/O unit sizes: 2048/2048, sub-page size 2048
[    9.647496] ubi1: VID header offset: 2048 (aligned 2048), data offset: 4096
[    9.654439] ubi1: good PEBs: 32, bad PEBs: 0, corrupted PEBs: 0
[    9.660341] ubi1: user volume: 1, internal volumes: 1, max. volumes count: 128
[    9.667540] ubi1: max/mean erase counter: 3/1, WL threshold: 4096, image sequence number: 4163859591
[    9.676648] ubi1: available PEBs: 0, total reserved PEBs: 32, PEBs reserved for bad PEB handling: 10
[    9.685761] ubi1: background thread "ubi_bgt1d" started, PID 905
[    9.692233] UBIFS (ubi1:0): Mounting in unauthenticated mode
[    9.697936] UBIFS (ubi1:0): background thread "ubifs_bgt1_0" started, PID 912
[    9.741978] UBIFS (ubi1:0): recovery needed
[    9.840658] UBIFS (ubi1:0): recovery completed
[    9.845124] UBIFS (ubi1:0): UBIFS: mounted UBI device 1, volume 0, name "tp_data"
[    9.852593] UBIFS (ubi1:0): LEB size: 126976 bytes (124 KiB), min./max. I/O unit sizes: 2048 bytes/2048 bytes
[    9.862486] UBIFS (ubi1:0): FS size: 1269760 bytes (1 MiB, 10 LEBs), journal size 888833 bytes (0 MiB, 5 LEBs)
[    9.872461] UBIFS (ubi1:0): reserved for root: 59973 bytes (58 KiB)
[    9.878714] UBIFS (ubi1:0): media format: w5/r0 (latest is w5/r0), UUID 6DB1F0D2-E677-4868-8BCB-866458EA1449, small LPT model
[    9.896568] ubi2: attaching mtd4
[    9.945746] ubi2: scanning is finished
[    9.957735] ubi2: attached mtd4 (name "userconfig", size 8 MiB)
[    9.963649] ubi2: PEB size: 131072 bytes (128 KiB), LEB size: 126976 bytes
[    9.970507] ubi2: min./max. I/O unit sizes: 2048/2048, sub-page size 2048
[    9.977272] ubi2: VID header offset: 2048 (aligned 2048), data offset: 4096
[    9.984216] ubi2: good PEBs: 64, bad PEBs: 0, corrupted PEBs: 0
[    9.990119] ubi2: user volume: 2, internal volumes: 1, max. volumes count: 128
[    9.997317] ubi2: max/mean erase counter: 3/1, WL threshold: 4096, image sequence number: 1377134154
[   10.006426] ubi2: available PEBs: 0, total reserved PEBs: 64, PEBs reserved for bad PEB handling: 20
[   10.015542] ubi2: background thread "ubi_bgt2d" started, PID 919
[   10.022217] UBIFS (ubi2:0): Mounting in unauthenticated mode
[   10.027918] UBIFS (ubi2:0): background thread "ubifs_bgt2_0" started, PID 925
[   10.072138] UBIFS (ubi2:0): recovery needed
[   10.209346] UBIFS (ubi2:0): recovery completed
[   10.213832] UBIFS (ubi2:0): UBIFS: mounted UBI device 2, volume 0, name "user_data1"
[   10.221565] UBIFS (ubi2:0): LEB size: 126976 bytes (124 KiB), min./max. I/O unit sizes: 2048 bytes/2048 bytes
[   10.231460] UBIFS (ubi2:0): FS size: 1269760 bytes (1 MiB, 10 LEBs), journal size 1396736 bytes (1 MiB, 11 LEBs)
[   10.241620] UBIFS (ubi2:0): reserved for root: 0 bytes (0 KiB)
[   10.247438] UBIFS (ubi2:0): media format: w4/r0 (latest is w5/r0), UUID EB768FE5-2A8C-4A68-974D-3A65D33583E4, small LPT model
[   10.266100] UBIFS (ubi2:1): Mounting in unauthenticated mode
[   10.271819] UBIFS (ubi2:1): background thread "ubifs_bgt2_1" started, PID 931
[   10.316080] UBIFS (ubi2:1): recovery needed
[   10.419256] UBIFS (ubi2:1): recovery completed
[   10.423738] UBIFS (ubi2:1): UBIFS: mounted UBI device 2, volume 1, name "user_data2"
[   10.431467] UBIFS (ubi2:1): LEB size: 126976 bytes (124 KiB), min./max. I/O unit sizes: 2048 bytes/2048 bytes
[   10.441359] UBIFS (ubi2:1): FS size: 1269760 bytes (1 MiB, 10 LEBs), journal size 1396736 bytes (1 MiB, 11 LEBs)
[   10.451509] UBIFS (ubi2:1): reserved for root: 0 bytes (0 KiB)
[   10.457323] UBIFS (ubi2:1): media format: w4/r0 (latest is w5/r0), UUID 299B3DFC-2931-44A5-8204-3F6EE251F552, small LPT model
[   10.497605] liblog: module license 'unspecified' taints kernel.
[   10.503536] Disabling lock debugging due to kernel taint
[   10.515487] ntfs: driver 2.1.32 [Flags: R/O MODULE].
[   10.525354] tun: Universal TUN/TAP device driver, 1.6
[   10.531953] sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
[   10.541244] Ethernet Channel Bonding Driver: v3.7.1 (April 27, 2011)
[   10.555852] mediatek_soc_hnat 15100000.hnat: wan = eth1
[   10.561104] mediatek_soc_hnat 15100000.hnat: lan = lan
[   10.566226] mediatek_soc_hnat 15100000.hnat: ppd = eth0
[   10.571438] mediatek_soc_hnat 15100000.hnat: gmac num = 2
[   10.576819] mediatek_soc_hnat 15100000.hnat: ppe num = 1
[   10.582528] mediatek_soc_hnat 15100000.hnat: PPE0 entry number = 8192
[   10.589455] mediatek_soc_hnat 15100000.hnat: PPE0 hwnat start
[   10.595238] hnat roaming work enable
[   10.602122] QCA Hy-Fi multicast installation successfully
[   10.849339] PPP generic driver version 2.4.2
[   10.854156] NET: Registered protocol family 24
[   10.859117] PPTP driver version 0.8.5
[   13.990882] fuse: init (API version 7.31)
[   14.751153] conninfra@(mtk_conninfra_drv_init:644) Before platform_driver_register
[   14.758823] Get Index(0-TOP_MISC_BASE) phy_addr(0x11d10000) vir_addr=(0xffffffc010afd000) size=(0x1000)
[   14.768190] Get Index(1-TOPRGU_BASE) phy_addr(0x1001c000) vir_addr=(0xffffffc010b15000) size=(0x1000)
[   14.777390] Get Index(2-GPIO_BASE) phy_addr(0x1001f000) vir_addr=(0xffffffc010b2d000) size=(0x1000)
[   14.786416] Get Index(3-IOCFG_TR_BASE) phy_addr(0x11f00000) vir_addr=(0xffffffc010b3d000) size=(0x1000)
[   14.795786] Get Index(4-IOCFG_TL_BASE) phy_addr(0x11f10000) vir_addr=(0xffffffc010b3f000) size=(0x1000)
[   14.805157] Get Index(5-INFRACFG_AO_BASE) phy_addr(0x10003000) vir_addr=(0xffffffc010b41000) size=(0x1000)
[   14.814790] Get Index(6-CONN_INFRA_CFG_BASE) phy_addr(0x18001000) vir_addr=(0xffffffc010b43000) size=(0x1000)
[   14.824680] Get Index(7-CONN_INFRA_SYSRAM_BASE) phy_addr(0x18050000) vir_addr=(0xffffffc010b45000) size=(0x1000)
[   14.834830] Get Index(8-CONN_INFRA_CLKGEN_ON_TOP_BASE) phy_addr(0x18009000) vir_addr=(0xffffffc010b4d000) size=(0x1000)
[   14.845586] Get Index(9-CONN_HOST_CSR_TOP_BASE) phy_addr(0x18060000) vir_addr=(0xffffffc010b4f000) size=(0x1000)
[   14.855737] Get Index(10-CONN_INFRA_BUS_CR_BASE) phy_addr(0x1800e000) vir_addr=(0xffffffc010b51000) size=(0x1000)
[   14.865973] Get Index(11-CONN_INFRA_RGU_BASE) phy_addr(0x18000000) vir_addr=(0xffffffc010b53000) size=(0x1000)
[   14.875949] Get Index(12-CONN_WT_SLP_CTL_REG_BASE) phy_addr(0x18005000) vir_addr=(0xffffffc010b55000) size=(0x1000)
[   14.886359] Get Index(13-INST2_CONN_WT_SLP_CTL_REG_BASE) phy_addr(0x18085000) vir_addr=(0xffffffc010b5d000) size=(0x1000)
[   14.897289] Get Index(14-CONN_RF_SPI_MST_REG_BASE) phy_addr(0x18004000) vir_addr=(0xffffffc010bad000) size=(0x1000)
[   14.907699] Get Index(15-INST2_CONN_RF_SPI_MST_REG_BASE) phy_addr(0x18084000) vir_addr=(0xffffffc010be2000) size=(0x1000)
[   14.918631] Get Index(16-CONN_SEMAPHORE_BASE) phy_addr(0x18070000) vir_addr=(0xffffffc010e40000) size=(0x10000)
[   14.928695] Get Index(17-CONN_AFE_CTL_BASE) phy_addr(0x18003000) vir_addr=(0xffffffc010be4000) size=(0x1000)
[   14.938499] Get Index(18-CONN_AFE_CTL_2ND_BASE) phy_addr(0x18083000) vir_addr=(0xffffffc010bed000) size=(0x1000)
[   14.948650] Get Index(19-WF_TOP_SLPPROT_ON_BASE) phy_addr(0x184c0000) vir_addr=(0xffffffc010e60000) size=(0x10000)
[   14.958974] Get Index(20-WF_TOP_CFG_BASE) phy_addr(0x184b0000) vir_addr=(0xffffffc010bf5000) size=(0x1000)
[   14.968603] Get Index(21-WF_MCU_CONFIG_LS_BASE) phy_addr(0x184f0000) vir_addr=(0xffffffc010bf7000) size=(0x1000)
[   14.978753] Get Index(22-WF_MCU_BUS_CR_BASE) phy_addr(0x18400000) vir_addr=(0xffffffc010bf9000) size=(0x1000)
[   14.988644] Get Index(23-WF_MCUSYS_INFRA_BUS_FULL_U_DEBUG_CTRL_AO_BASE) phy_addr(0x18500000) vir_addr=(0xffffffc010bfb000) size=(0x1000)
[   15.000874] Get Index(24-WF_TOP_CFG_ON_BASE) phy_addr(0x184c0000) vir_addr=(0xffffffc010e80000) size=(0x10000)
[   15.010859] conninfra@(FlashRead:106) Can't get mtd device!
[   15.016411] [1;31meeFlashId(0x0) isn't match with 0x7986![0m
[   15.016413] [1;31mFlash content is empty, so use AX6000 configuration by default![0m
[   15.022229] conninfra@(consys_plt_hw_init:230) adie_cfg_type = 2, one_adie_dbdc = 0
[   15.037759] [emi_mng_init] gConEmiPhyBase = [0x4fc00000] size = [0x100000] fw size = [0x100000] ops=[00000000dfbc743c]
[   15.048512] conninfra@(mtk_conninfra_drv_init:650) After platform_driver_register
[   15.056317] ConnInfra Dev: init (0)
[   15.061298] conninfra@(_consys_polling_chipid_int:459) Read CONNSYS HW IP version successfully! (0x02070000)
[   15.071110] conninfra@(consys_bus_clock_ctrl:485) drv=[4] conninfra_bus_clock_wpll=[0]->[16] enable
[   15.080137] conninfra@(connsys_d_die_cfg:530) D-die efuse: 0x00000000
[   15.087590] conninfra@(_connsys_a_die_chip_id_confirm:1915) adie_idx[0]: A-die CHIP ID = 0x7975, HW Version = 0x8a02
[   15.099101] conninfra@(_connsys_a_die_chip_id_confirm:1915) adie_idx[1]: A-die CHIP ID = 0x7975, HW Version = 0x8a02
[   15.111614] conninfra@(consys_spi_write_nolock:670) addr = 0x0a00, val = 0xffffffff
[   15.121263] conninfra@(consys_spi_write_nolock:670) addr = 0x0a1c, val = 0x30000a8a
[   15.129906] conninfra@(consys_spi_write_nolock:670) addr = 0x0144, val = 0x00000002
[   15.139551] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x00000000
[   15.149194] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x43a60000
[   15.162867] conninfra@(connsys_a_die_efuse_read_nolock:906) efuse = [0x0000ae02, 0x00c00100, 0x008400b4, 0x00000000]
[   15.174369] conninfra@(consys_spi_write_nolock:670) addr = 0x0144, val = 0x00000002
[   15.184018] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x23a60036
[   15.193667] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x63a70036
[   15.207344] conninfra@(connsys_a_die_efuse_read_nolock:906) efuse = [0x0000ae02, 0x00c00100, 0x008400b4, 0x00000000]
[   15.218846] conninfra@(consys_spi_write_nolock:670) addr = 0x0144, val = 0x00000002
[   15.228490] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x23a70036
[   15.238133] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x63a50036
[   15.251811] conninfra@(connsys_a_die_efuse_read_nolock:906) efuse = [0x0000ae02, 0x00c00100, 0x008400b4, 0x00000000]
[   15.262306] conninfra@(_connsys_a_die_form_patch_7975:1164) Form 7975 adie Patch
[   15.270689] conninfra@(consys_spi_write_nolock:670) addr = 0x0348, val = 0x00000002
[   15.279331] conninfra@(consys_spi_write_nolock:670) addr = 0x0378, val = 0x00000002
[   15.287968] conninfra@(consys_spi_write_nolock:670) addr = 0x03a8, val = 0x00000002
[   15.296610] conninfra@(consys_spi_write_nolock:670) addr = 0x03d8, val = 0x00000002
[   15.305252] conninfra@(consys_spi_write_nolock:670) addr = 0x0a1c, val = 0x30000aaa
[   15.313894] conninfra@(consys_spi_write_nolock:670) addr = 0x0a84, val = 0x8470008a
[   15.322538] conninfra@(consys_spi_write_nolock:670) addr = 0x0074, val = 0x00000002
[   15.331181] conninfra@(consys_spi_write_nolock:670) addr = 0x0aa4, val = 0x01001fc0
[   15.339823] conninfra@(consys_spi_write_nolock:670) addr = 0x0070, val = 0x00000005
[   15.348461] conninfra@(consys_spi_write_nolock:670) addr = 0x0344, val = 0x00000088
[   15.357097] conninfra@(consys_spi_write_nolock:670) addr = 0x0374, val = 0x00000088
[   15.365739] conninfra@(consys_spi_write_nolock:670) addr = 0x03a4, val = 0x00000088
[   15.374381] conninfra@(consys_spi_write_nolock:670) addr = 0x03d4, val = 0x00000088
[   15.383023] conninfra@(consys_spi_write_nolock:670) addr = 0x0a80, val = 0x44d07000
[   15.391665] conninfra@(consys_spi_write_nolock:670) addr = 0x0a88, val = 0x3900aaaa
[   15.400306] conninfra@(consys_spi_write_nolock:670) addr = 0x02c4, val = 0x00000000
[   15.408944] conninfra@(consys_spi_write_nolock:670) addr = 0x02c8, val = 0x00000072
[   15.418591] conninfra@(consys_spi_write_nolock:670) addr = 0x0a00, val = 0xffffffff
[   15.428235] conninfra@(consys_spi_write_nolock:670) addr = 0x0a1c, val = 0x30000a8a
[   15.436876] conninfra@(consys_spi_write_nolock:670) addr = 0x0144, val = 0x00000002
[   15.446524] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x00000000
[   15.456173] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x43a60000
[   15.469849] conninfra@(connsys_a_die_efuse_read_nolock:906) efuse = [0x0000ac02, 0x00c00100, 0x008400bd, 0x00000000]
[   15.481350] conninfra@(consys_spi_write_nolock:670) addr = 0x0144, val = 0x00000002
[   15.490998] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x23a60036
[   15.500646] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x63a70036
[   15.514318] conninfra@(connsys_a_die_efuse_read_nolock:906) efuse = [0x0000ac02, 0x00c00100, 0x008400bd, 0x00000000]
[   15.525819] conninfra@(consys_spi_write_nolock:670) addr = 0x0144, val = 0x00000002
[   15.535468] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x23a70036
[   15.545116] conninfra@(consys_spi_write_nolock:670) addr = 0x0148, val = 0x63a50036
[   15.558792] conninfra@(connsys_a_die_efuse_read_nolock:906) efuse = [0x0000ac02, 0x00c00100, 0x008400bd, 0x00000000]
[   15.569287] conninfra@(_connsys_a_die_form_patch_7975:1164) Form 7975 adie Patch
[   15.577665] conninfra@(consys_spi_write_nolock:670) addr = 0x0348, val = 0x00000002
[   15.586309] conninfra@(consys_spi_write_nolock:670) addr = 0x0378, val = 0x00000002
[   15.594951] conninfra@(consys_spi_write_nolock:670) addr = 0x03a8, val = 0x00000002
[   15.603593] conninfra@(consys_spi_write_nolock:670) addr = 0x03d8, val = 0x00000002
[   15.612235] conninfra@(consys_spi_write_nolock:670) addr = 0x0a1c, val = 0x30000aaa
[   15.620877] conninfra@(consys_spi_write_nolock:670) addr = 0x0a84, val = 0x8470008a
[   15.629518] conninfra@(consys_spi_write_nolock:670) addr = 0x0074, val = 0x00000002
[   15.638155] conninfra@(consys_spi_write_nolock:670) addr = 0x0aa4, val = 0x01001fc0
[   15.646797] conninfra@(consys_spi_write_nolock:670) addr = 0x0070, val = 0x00000005
[   15.655439] conninfra@(consys_spi_write_nolock:670) addr = 0x0344, val = 0x00000088
[   15.664080] conninfra@(consys_spi_write_nolock:670) addr = 0x0374, val = 0x00000088
[   15.672722] conninfra@(consys_spi_write_nolock:670) addr = 0x03a4, val = 0x00000088
[   15.681363] conninfra@(consys_spi_write_nolock:670) addr = 0x03d4, val = 0x00000088
[   15.690005] conninfra@(consys_spi_write_nolock:670) addr = 0x0a80, val = 0x44d07000
[   15.698641] conninfra@(consys_spi_write_nolock:670) addr = 0x0a88, val = 0x3900aaaa
[   15.707279] conninfra@(consys_spi_write_nolock:670) addr = 0x02c4, val = 0x00000000
[   15.715920] conninfra@(consys_spi_write_nolock:670) addr = 0x02c8, val = 0x00000072
[   15.724564] conninfra@(consys_spi_write_nolock:670) addr = 0x0070, val = 0x80000000
[   15.734163] conninfra@(consys_spi_write_nolock:670) addr = 0x0070, val = 0x00000005
[   15.742806] conninfra@(consys_spi_write_nolock:670) addr = 0x0070, val = 0x80000000
[   15.752406] conninfra@(consys_spi_write_nolock:670) addr = 0x0070, val = 0x00000005
[   15.760988] conninfra@(_consys_polling_chipid_int:459) Read CONNSYS HW IP version successfully! (0x02070000)
[   15.770792] conninfra@(_consys_hw_conninfra_wakeup:470) conninfra_wakeup refcnt=[0]->[1] wakeup!!
[   15.779651] conninfra@(_consys_hw_conninfra_sleep:490) conninfra_sleep refcnt=[1]->[0] sleep!!
[   15.788237] conninfra@(opfunc_power_on_internal:241) [Conninfra Pwr On] BT=[0] FM=[0] GPS=[0] WF=[0] CONNINFRA=[1]
[   17.574240] conninfra@(conninfra_pwr_on:120) [conninfra_pwr_on] drv=[3]
[   17.581796] conninfra@(_consys_polling_chipid_int:459) Read CONNSYS HW IP version successfully! (0x02070000)
[   17.599644] [1;32mAdie Type: 0x7975[0m
[   17.599645] [1;32mSKU Type: 6000[0m
[   17.603553] conninfra@(opfunc_power_on_internal:241) [Conninfra Pwr On] BT=[0] FM=[0] GPS=[0] WF=[1] CONNINFRA=[1]
[   17.617608] WiFi@C12L1,get_dbg_setting_by_profile() 4230: debug level setting=INDEX0_debug_level not found!!
[   17.627428] WiFi@C12L1,get_dbg_setting_by_profile() 4253: debug option setting=INDEX0_debug_option not found!!
[   17.637459] mt_rbus 0000:00:00.0: runtime IRQ mapping not provided by arch
[   17.651040] mt7986_init()-->
[   17.653933] [1;33m sku:0xd, ADIE=MT7975 [0m
[   17.658272] mt7986_init(), vlan2ethctrl:1 
[   17.662361] <--mt7986_init()
[   17.665252] RtmpRaDevCtrlInit(): MCS_LUT not used becasue MacTb size(544) > 128!
[   17.672731] WiFi@C12L1,match_index_by_chipname() 507: not find chip name INDEX1
[   17.680037] WiFi@C12L1,match_index_by_chipname() 507: not find chip name INDEX2
[   17.687335] WiFi@C12L1,match_index_by_chipname() 507: not find chip name INDEX3
[   17.694639] WiFi@C12L1,match_index_by_chipname() 507: not find chip name INDEX4
[   17.701943] WiFi@C12L1,match_index_by_chipname() 507: not find chip name INDEX5
[   17.709245] WiFi@C12L1,match_index_by_chipname() 507: not find chip name INDEX6
[   17.716544] WiFi@C12L1,match_index_by_chipname() 507: not find chip name INDEX7
[   17.723846] WiFi@C12L1,match_index_by_chipname() 507: not find chip name INDEX8
[   17.731148] WiFi@C12L1,match_index_by_chipname() 507: not find chip name INDEX9
[   17.738438] WiFi@C12L2,match_index_by_chipname() 512: Not find slot id, back to old rule.
[   17.746681] msta_ifname not found
[   17.984459] warp_module_init(): module init and register callback for warp
[   17.991358] create warp_ctrl ok!!!
[   17.994780] wed_get_slot_map(): assign slot_id:0 for entry: 0!
[   18.000618] wed_get_slot_map(): assign slot_id:1 for entry: 1!
[   18.198575] mtk_warp_proxy: module uses symbols from proprietary module mt_wifi, inheriting taint.
reloadprofile() begin
reloadprofile() end
reloadconfig() begin
get_upgrade_level() begin
/tmp/default-config/config/upgrade: No such file or directory
============> upgrade_level = 0
get_upgrade_level() end
get_upgrade_level() begin
/tmp/user-config/config/upgrade: No such file or directory
============> upgrade_level = 0
get_upgrade_level() end
!!!!===> not do resetandmergeconfig .....
reloadconfig() end
mergeconfigbycountry() end
device is not production models, do nothing!!!
loadRepeaterProductInfo() end
[GPIOD][gpio_create_ibus_thread:35]create ibus thread successfully

[   19.866771] mtk_soc_eth 15100000.ethernet eth1: configuring for fixed/2500base-x link mode
[   19.900799] br-lan: port 1(eth1) entered blocking state
[   19.906029] br-lan: port 1(eth1) entered disabled state
[   19.911418] device eth1 entered promiscuous mode
[   19.927460] mtk_soc_eth 15100000.ethernet eth0: configuring for fixed/2500base-x link mode
[   19.935913] mtk_soc_eth 15100000.ethernet eth0: Link is Up - 2.5Gbps/Full - flow control rx/tx
[   19.937854] device eth0 entered promiscuous mode
[   19.949739] mt7530 mdio-bus:1f lan0: configuring for phy/gmii link mode
[   19.956823] 8021q: adding VLAN 0 to HW filter on device lan0
[   19.966879] mt7530 mdio-bus:1f lan1: configuring for phy/gmii link mode
[   19.973992] 8021q: adding VLAN 0 to HW filter on device lan1
[   19.984127] mt7530 mdio-bus:1f lan2: configuring for phy/gmii link mode
[   19.991246] 8021q: adding VLAN 0 to HW filter on device lan2
[   20.001227] mt7530 mdio-bus:1f lan3: configuring for phy/gmii link mode
[   20.008412] 8021q: adding VLAN 0 to HW filter on device lan3
[   20.017185] br-lan: port 2(lan0) entered blocking state
[   20.022426] br-lan: port 2(lan0) entered disabled state
[   20.027948] device lan0 entered promiscuous mode
[   20.035251] br-lan: port 3(lan1) entered blocking state
[   20.040499] br-lan: port 3(lan1) entered disabled state
[   20.046130] device lan1 entered promiscuous mode
[   20.053441] br-lan: port 4(lan2) entered blocking state
[   20.058658] br-lan: port 4(lan2) entered disabled state
[   20.064536] device lan2 entered promiscuous mode
[   20.071939] br-lan: port 5(lan3) entered blocking state
[   20.077155] br-lan: port 5(lan3) entered disabled state
[   20.083212] device lan3 entered promiscuous mode
tz isGMT-01:00	
[SMARTIPD] [smartip_create_ibus_[   20.253818] [DHCP_FILTER] set dhcp_filter sysmode  0
thread 40] create ibus thread successfully
[SMARTIPD] [smartip_generate_udhcpd_cfg 2024] smartip generate udhcpd conf read success. readcount = 0

[SMARTIPD] [smartip_start_process 787] smartipd start udhcpd...........
server: 192.168.0.100[SMARTIPD] [onemesh_get_mesh_enable 999] Parse JSON failed!
@@@@@@@@ap follow sta ...
wifid[mtk_init_platform:7353]: mtk init platform start.
wifid[wifi_create_ibus_thread:40]: Create ibus thread successfully
[SMARTIPD] [smartip_receive_event 66] smartipd received action: 2, prelink_status: 0
[SMARTIPD] [smartip_receive_event 119] in ibus action wifi: set prelink_status to FALSE

[SMARTIPD] [smartip_receive_event 140] smartip_receive_event handle out

wifid[wifi_exec_cmd:1957]: iwpriv apclix0 set DfsPrelink=0
wifid[wifi_get_production_models:381]: get char 0 

wifid[wifi_exec_cmd:1957]: mkdir -p /tmp/wireless/
wifid[wifi_exec_cmd:1957]: cp /et[   20.676532] WiFi@C01L2,RTMP_COM_IoctlHandle() 1505: -> CMD_RTPRIV_IOCTL_VIRTUAL_INF_INIT
c/wireless/2G_Pr[   20.685838] msta_ifname not found
ofile.dat /tmp/w[   20.690936] warp_cap_support(): chip_id=7986 is in WHNAT support list
ireless/2G_Profi[   20.698116] warp_entry_acquire(): bus slot:0, hook to WARP, entry id=0, entry: 00000000ea4660be
le.dat
wifid[wi[   20.708183] client_hif_specific_get(): hw->base_phy_addr 0x18000000!!
fi_exec_cmd:1957[   20.716081] warp_entry_get_by_pdev(): return entry[0] 
]: cp /etc/wirel[   20.722584] warp_msg_register():warp,pid,1813,id,0,wed_idx,0
ess/5G_Profile.d[   20.729578] wed_init(0): res_start = 0x15010000, size = 1000, irq=6,base_addr=0xffffffc010b5f000
at /tmp/wireless[   20.740063] wed_ring_init(): wed tx ring init result = 0
/5G_Profile.dat
[   20.746804] wed_ring_init(): wed rx ring init result = 0

wifid[_init_config_file:2512]: 

++++++++l_mtk.DFS_support = TRUE+++++++++
scanTime_2G:15 sc[   20.761230] create wed ok!!!
anTime_5G:18

[   20.764242] wdma_init(): wdma(0) base_addr=0xffffffc010b61800, base_phy_addr=0x15104800
wifid[wifi_exec_[   20.774311] warp_woif_bus_init_hw(): bus:000000001730976c, txring: 20, pa: 0x00000000578c8000, 0x00000000578c8000, 24, 28, 3c
cmd:1957]: wifi [   20.786256] warp_woif_bus_init_hw(): bus:000000001730976c, rxring: 50, pa: 0x00000000578ca000, 0x00000000578ca000, 54, 58, 6c
forcesetscantime[   20.798939] get_region_info(): wed_idx = 0, base_addr = 0x0000000047ab4c50, size = 0x40000, res.start = 0x4fd00000, shared:0
 15 18
wifid[wi[   20.811500] get_region_info(): wed_idx = 0, base_addr = 0x00000000780f927c, size = 0x8000, res.start = 0x151e0000, shared:0
fi_exec_cmd:1957[   20.824009] get_region_info(): wed_idx = 0, base_addr = 0x00000000f8d2edd5, size = 0x240000, res.start = 0x4fd80000, shared:1
]: ifconfig ra0 [   20.836642] get_region_info(): wed_idx = 0, base_addr = 0x000000009b2bd111, size = 0x1000, res.start = 0x15194000, shared:0
up
[   20.849119] warp_fwdl_mcu_mode(): mcu mode, need fwdl
[   20.891146] ########## WO Firmware ##########
[   20.895496] Chip ID: 0x0000
[   20.898274] ECO version:0
[   20.900904] Version: DEV_000000
[   20.904032] Build date: 20220819140108
[   20.907763] Total region:3
[   20.910462] #################################
[   20.914801] Parsing tailer region 0
[   20.918273] 	Target address: 0x151e0000
[   20.922102] 	Download size: 20400
[   20.925428] Parsing tailer region 1
[   20.928918] 	Target address: 0x4fd00000
[   20.932740] 	Download size: 87120
[   20.936105] Parsing tailer region 2
[   20.939587] 	Target address: 0x4fd80000
[   20.943409] 	Download size: 2283984
[   20.948505] warp_fwdl_ready_check_mcu_mode(): waiting for wocpu
[   20.970323] warp_fwdl_ready_check_mcu_mode(): wocpu is ready
[   20.975990] warp_woctrl_init_state(), wo_state: WO_STATE_ENABLE
[   20.981905] wo_proc_init done  000000007a98a48e
[   20.986419] wo_exep_proc_init done  000000007a98a48e
[   20.991398] wo_exception_init(0): exp log= 0x00000000e9587e53, phy_addr= 0x0000000055080000 size= 32768
[   21.000877] warp_probe(): probe fail since warp is full!
[   21.006181] warp_dev0: probe of 15010000.wed2 failed with error -1
[   21.016657] WiFi@C02L1,HwCtrlInit() 525: ser_init() return NDIS_STATUS_FAILURE
[   21.091941] WiFi@C12L1,RTMPWirelessModeCfg() 587: Init: BSS0 PhyMode=78
[   21.098548] WiFi@C12L1,RTMPWirelessModeCfg() 587: Init: BSS1 PhyMode=78
[   21.105142] WiFi@C12L1,RTMPWirelessModeCfg() 587: Init: BSS2 PhyMode=177
[   21.111833] WiFi@C12L1,RTMPWirelessModeCfg() 587: Init: BSS3 PhyMode=177
[   21.118801] WiFi@C23L2,auto_ch_select_set_cfg() 3481: BandIdx0, AutoChannelAtBootup=0, AutoChannelAlg = 0
[   21.128351] WiFi@C23L2,auto_ch_select_set_cfg() 3481: BandIdx1, AutoChannelAtBootup=0, AutoChannelAlg = 0
[   21.149350] WiFi@C12L1,SCS_Enable_set_cfg() 711: SCSEnable [0]= 0
[   21.155430] WiFi@C12L1,SCS_Enable_set_cfg() 711: SCSEnable [1]= 0
[   21.167427] I/F(ra0) GroupKey ReKeyInterval=0 seconds
[   21.172471] I/F(ra1) GroupKey ReKeyInterval=0 seconds
[   21.177502] I/F(ra2) GroupKey ReKeyInterval=0 seconds
[   21.182542] I/F(ra3) GroupKey ReKeyInterval=0 seconds
[   21.211088] WiFi@C15L1,Set_PMFMFPC_Proc() 1728: [PMF] :: apidx=0, Desired MFPC=0
[   21.218467] WiFi@C15L1,Set_PMFMFPC_Proc() 1728: [PMF] :: apidx=1, Desired MFPC=0
[   21.225840] WiFi@C15L1,Set_PMFMFPC_Proc() 1728: [PMF] :: apidx=2, Desired MFPC=0
[   21.233224] WiFi@C15L1,Set_PMFMFPC_Proc() 1728: [PMF] :: apidx=3, Desired MFPC=0
[   21.242547] WiFi@C12L1,RTMPSetProfileParameters() 8299: ra0, QoSMgmtCapa=0
[   21.249410] WiFi@C12L1,RTMPSetProfileParameters() 8299: (null), QoSMgmtCapa=0
[   21.256523] WiFi@C12L1,RTMPSetProfileParameters() 8299: (null), QoSMgmtCapa=0
[   21.263645] WiFi@C12L1,RTMPSetProfileParameters() 8299: (null), QoSMgmtCapa=0
[   21.275363] WiFi@C12L1,rtmp_read_ap_client_from_file() 1605: APCLI[0] ApCliMuMimoDlEnable = 0
[   21.283872] WiFi@C12L1,rtmp_read_ap_client_from_file() 1605: APCLI[1] ApCliMuMimoDlEnable = 0
[   21.292525] WiFi@C12L1,rtmp_read_ap_client_from_file() 1624: APCLI[0] ApCliMuMimoUlEnable = 0
[   21.301034] WiFi@C12L1,rtmp_read_ap_client_from_file() 1624: APCLI[1] ApCliMuMimoUlEnable = 0
[   21.315381] WiFi@C23L2,read_vht_param_from_file() 2670: [DFS_ZDFS_ENHANCE], Pre CH: 0,	Pre BW: 0,	Pre CAC done: 0
[   21.325625] WiFi@C23L2,read_vht_param_from_file() 2670: [DFS_ZDFS_ENHANCE], Pre CH: 0,	Pre BW: 0,	Pre CAC done: 0
[   21.393579] BandIdx = 0, tpcDutyInitFlag=1
[   21.397662] BandIdx = 1, tpcDutyInitFlag=1
[   21.401921] BandIdx = 0, tpcnitFlag=1
[   21.405566] BandIdx = 1, tpcnitFlag=1
[   21.409390] BandIdx = 0, tpcDuty=100-75-50-25
[   21.413730] BandIdx = 1, tpcDuty=100-75-50-25
[   21.418235] BandIdx = 0, thermalHighEn=1
[   21.422150] BandIdx = 1, thermalHighEn=1
[   21.426225] BandIdx = 0, thermalHighCritera=120
[   21.430745] BandIdx = 1, thermalHighCritera=120
[   21.435425] BandIdx = 0, thermalLowEn=1
[   21.439252] BandIdx = 1, thermalLowEn=1
[   21.443239] BandIdx = 0, thermalLowCritera=115
[   21.447664] BandIdx = 1, thermalLowCritera=115
[   21.452267] BandIdx = 0, thermalRecheckTime=5
[   21.456605] BandIdx = 1, thermalRecheckTime=5
[   21.461120] BandIdx = 0, thermalRFOffEn=0
[   21.465111] BandIdx = 1, thermalRFOffEn=0
[   21.469280] BandIdx = 0, thermalRFOffCriteria=125
[   21.473965] BandIdx = 1, thermalRFOffCriteria=125
[   21.478829] BandIdx = 0, thermalUcType=1
[   21.482733] BandIdx = 1, thermalUcType=1
[   21.488126] wdma_dma_ctrl(): WDMA_GLO_CFG=50404e70, txrx = 0
[   21.499299] wdma_dma_ctrl(): WDMA_GLO_CFG=40804e70, txrx = 0
[   21.512650] warp_tx_ring_init_hw(): configure ring 0 setting
[   21.518322] warp_tx_ring_init_hw(): wed:00000000d8b4af0d wifi:000000008460243c: 24420=56d58000,24424=2048,24428=0
[   21.528566] warp_tx_ring_init_hw(): configure ring 1 setting
[   21.534210] warp_tx_ring_init_hw(): wed:00000000d8b4af0d wifi:000000008460243c: 24430=56c78000,24434=2048,24438=0
[   21.544565] set_rrocfg,0
[   21.569955] wdma_dma_ctrl(): WDMA_GLO_CFG=40804e74, txrx = 3
[   21.575609] [1;33m sku:0xd, ADIE=MT7975 [0m
[   21.579957] using E1 ROM patch
[   21.582999] using E1 RAM
[   21.585523] current sync CR = 0x1
[   21.588830] 	Built date: 20220819135650a
[   21.592742] 	Platform: ALPS
[   21.595522] 	HW/SW version: 0x8a108a10
[   21.599262] 	Patch version: 0xffffffff
[   21.608797] current sync CR = 0x1
[   21.612098] 	Chip ID: 0x0f
[   21.614790] 	Eco version: 0x00
[   21.617830] 	Region number: 0x0b
[   21.621053] 	Format version: 0x02
[   21.624353] 	Format flag: 0x01
[   21.627391] 	Ram version: ____000000
[   21.630966] 	Built date: 20220819135752
[   21.634792] 	Common crc: 0x3d2c5016
[   21.638268] 	Release info: header tag = 0, total length = 108
[   21.644017] 	tag 1, padding length = 1, tag length = 103
[   21.649320] 	payload: t-neptune-main-2111-7986-imp_backup_wk2213-MT7986_E1_ASIC_ROM_RAM_REBB_IMP_BACKUP_WK2213-20220819135053
[   21.676438] 	Chip ID: 0x00
[   21.679159] 	Eco version: 0x00
[   21.682203] 	Region number: 0x03
[   21.685416] 	Format version: 0x02
[   21.688730] 	Format flag: 0x01
[   21.691778] 	Ram version: ____000000
[   21.695342] 	Built date: 20220823133834
[   21.699188] 	Common crc: 0xe10f2488
[   21.702665] 	Release info: header tag = 0, total length = 0
[   21.801287] current sync CR = 0x7
[   21.804610] efuse_probe: efuse = deaddead
[   21.808691] rtmp_ee_load_from_bin::bin FileName = /tp_data/MT7986_EEPROM.bin, bin bak = /user_data1/MT7986_EEPROM.bin, use caled e2p file
[   21.823751] Load EEPROM Buffer from /tp_data/MT7986_EEPROM.bin
[   21.829621] rtmp_btn_load_from_bin::bin FileName = /tp_data/MT7986_BTNFLAG.bin, bin bak = /user_data1/MT7986_BTNFLAG.bin
[   21.840504] WiFi@C01L1,RtmpOSFileOpen() 1008: Error 2 opening /tp_data/MT7986_BTNFLAG.bin
[   21.848688] WiFi@C02L1,rtmp_btn_load_from_bin() 2362: Error opening ,use backup btn file /user_data1/MT7986_BTNFLAG.bin
[   21.859502] WiFi@C01L1,RtmpOSFileOpen() 1008: Error 2 opening /user_data1/MT7986_BTNFLAG.bin
[   21.867947] [1;33m sku:0xd, ADIE=MT7975 [0m
[   21.872342] [1;33m sku:0xd, ADIE=MT7975 [0m
[   21.945899] [1;33m sku:0xd, ADIE=MT7975 [0m
[   22.001031] WiFi@C02L1,is_cal_free_ic() 1022: [41m iCal free[m
[   22.007034] [1;33m sku:0xd, ADIE=MT7975 [0m
table: 0x3ddce320	
on	
off	
wpsd[wps_create_ibus_thread:43]: create ibus thread successfully
wpsd[wpsd_backup_run_mode:1408]: DUT RUNNING AS A SUPER REPEATER!!!
wpsd[wpsd_backup_RFBand:1331]: RFBand: 2

wpsd[wpsd_check_available_status:1186]: enabled: 1, apWpsAvailable: 3, staWpsAvailable: 3, pinEnabled: 1, pinCode:98322961

!!!!!!!!!!!!!!!!!!!cloud_brd start!!!!!
!!!!!!!!!!!!!!!!!!cloud-client start!!!
[cloud_cli.c]: main()->cloud client start.
<APSD_INFO>[main, 105] Loop avoidance initialized!
<APSD_INFO>[main, 105] Path Selection initialized!
<APSD_INFO>[path_selection_event_cb, 365] get event: 0

<APSD_INFO>[loop_avoidance_configure_bridge_with_mode, 115] configure hyfi bridge with mode RE
<APSD_ERR>[apsd_get_wireless_link_state, 292] parse /tmp/wifi_runtime_info.2g failed
<APSD_ERR>[apsd_get_wireless_link_state, 292] parse /tmp/wifi_runtime_info.5g failed
device not binded
event:11
ubus call cloud_client response is  0
[   25.393610] WiFi@C17L1,RTMPReadTxPwrPerRate() 450: (450): Don't Support this now!
[   25.401115] WiFi@C02L0,rc_radio_init() 706: radio_ctrl=00000000629206f2,Band=0,rfcap=1,channel=1,PhyMode=64 extCha=0xf
[   25.411816] WiFi@C02L0,rc_radio_init() 706: radio_ctrl=00000000fbd6ad78,Band=1,rfcap=2,channel=36,PhyMode=128 extCha=0xf
[   25.422713] WiFi@C01L1,AntCfgInit() 3089: Not support for HIF_MT yet!
[   25.442044] ch_switch_monitor_state_machine_init:: enter
[   25.447345] ch_switch_monitor_cfg_reset:: enter
[   25.451877] ch_switch_monitor_cfg_reset:: enter
[   25.456690] WiFi@C17L1,tx_pwr_comp_init() 641: NotSupportYet!
[   25.462481] WiFi@C12L1,SCS_init() 880: INIT SCSEnable [0]= 0
[   25.468133] WiFi@C12L1,SCS_init() 887: INIT SCSEnable [1]= 0
[   25.473886] WiFi@C03L1,MtCmdSetMacTxRx() 10214: (ret = 0)
[   25.479366] WiFi@C03L1,MtCmdSetMacTxRx() 10214: (ret = 0)
[   25.484913] WiFi@C14L1,IPMacTable_init() 348: IPMacTable already inited!
[   25.511734] wdma_dma_ctrl(): WDMA_GLO_CFG=58804e75, txrx = 3
[   25.517470] WiFi@C03L1,MtCmdSetMacTxRx() 10214: (ret = 0)
[   25.522955] WiFi@C03L1,MtCmdSetMacTxRx() 10214: (ret = 0)
[   25.528407] WiFi@C00L1,set_ack_timeout_mode_byband() 13292: CTS/ACK Timeout Range should between [0xFFFF:0]!!
[   25.538305] WiFi@C00L1,set_datcfg_ack_cts_timeout() 13380: DAT config band(0) cck_timeout Fail!
[   25.546979] WiFi@C00L1,set_ack_timeout_mode_byband() 13292: CTS/ACK Timeout Range should between [0xFFFF:0]!!
[   25.556874] WiFi@C00L1,set_datcfg_ack_cts_timeout() 13391: DAT config band(0) ofdm_timeout Fail!
[   25.565637] WiFi@C00L1,set_ack_timeout_mode_byband() 13292: CTS/ACK Timeout Range should between [0xFFFF:0]!!
[   25.575532] WiFi@C00L1,set_datcfg_ack_cts_timeout() 13402: DAT config band(0) ofdma_timeout Fail!
[   25.584382] WiFi@C00L1,set_ack_timeout_mode_byband() 13292: CTS/ACK Timeout Range should between [0xFFFF:0]!!
[   25.594275] WiFi@C00L1,set_datcfg_ack_cts_timeout() 13380: DAT config band(1) cck_timeout Fail!
[   25.602952] WiFi@C00L1,set_ack_timeout_mode_byband() 13292: CTS/ACK Timeout Range should between [0xFFFF:0]!!
[   25.612862] WiFi@C00L1,set_datcfg_ack_cts_timeout() 13391: DAT config band(1) ofdm_timeout Fail!
[   25.621630] WiFi@C00L1,set_ack_timeout_mode_byband() 13292: CTS/ACK Timeout Range should between [0xFFFF:0]!!
[   25.631528] WiFi@C00L1,set_datcfg_ack_cts_timeout() 13402: DAT config band(1) ofdma_timeout Fail!
[   25.640452] mt_wifi_init: Band = 0, dutyLv0 = 100, dutyLv1 = 75, dutyLv2 = 50, dutyLv3 = 25
[   25.648800] mt_wifi_init: Band = 0, high_en=1, high_thd = 120, low_en = 1, low_thd = 115, RFOFF_en = 0, RFOFF_th = 125, rec_timer = 5, Type = Duty Cycle
[   25.662475] mt_wifi_init: Band = 1, dutyLv0 = 100, dutyLv1 = 75, dutyLv2 = 50, dutyLv3 = 25
[   25.670820] mt_wifi_init: Band = 1, high_en=1, high_thd = 120, low_en = 1, low_thd = 115, RFOFF_en = 0, RFOFF_th = 125, rec_timer = 5, Type = Duty Cycle
[   25.685998] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
[   25.694156] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
[   25.901999] :(op = 1)
[   25.909044] WiFi@C00L1,FastPathCheckMIC() 1010: (ret = 0)(op:1)
[   25.914947] WiFi@C00L2,red_tail_drop_init() 83: red_tail_drop_init: available token:8192, whnat_en:1
[   25.924082] WiFi@C00L1,MtCmdCr4RedSet() 914: MtCmdCr4RedSet: (ret = 0)
[   25.930594] WiFi@C00L1,SendRedCmd() 1049: SendRedCmd:(ret = 1)
[   25.936552] WiFi@C01L2,RTMP_COM_IoctlHandle() 1482:  -> CMD_RTPRIV_IOCTL_VIRTUAL_INF_UP
[   25.944551] WiFi@C01L2,wifi_sys_open() 832: wdev idx = 0
[   25.950144] WiFi@C08L3,wifi_sys_op_lock() 791: wifi_sys_open+0x48/0x408 [mt_wifi](0): get wdev_op_lock.
[   25.959573] [Nop_List_Backup] NopList Channel count:0 <----
[   25.965160] WiFi@C23L1,DfsBuildChannelList() 3551: [RDM]: wdev is not 5G.
[   25.971954] [RadarStateCheck] Zero Wait State: 0
[   25.976655] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   25.985790] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=6, ctrl_ch2=0, cent_ch=8 DBDCIdx=0, ChBand=0, BW=1, TXStream=4, RXStream=4, scan(0)
[   26.240865] WiFi@C00L2,dump_ht_cap() 29: Caller: wlan_operate_init+0xac/0x120 [mt_wifi]
[   26.248885] wdev_attr_update(): wdevId0 = 30:de:4b:4e:e0:6b
[   26.254538] WiFi@C12L1,wifi_sys_open() 874: wifi_sys_open():band0 group_idx[0]=0, bw_grp=16 omac=0
[   26.263515] WiFi@C08L3,hw_ctrl_flow_v2_open() 13: wdev_idx=0
[   26.269337] WiFi@C08L3,wifi_sys_op_unlock() 806: release wdev_op_lock.
[   26.275863] WiFi@C02L2,AsicRadioOnOffCtrl() 2279: DbdcIdx=0 RadioOn
[   26.282669] mtk_band_steering_netlink_init 247 band steering netlink init ok

[   26.289826] WiFi@C00L2,ApAutoChannelAtBootUp() 301: ----------------->
[   26.296334] WiFi@C00L0,ApAutoChannelAtBootUp() 317: AutoChannelBootup[0] = 0
[   26.303465] WiFi@C03L1,MtCmdSetMacTxRx() 10214: (ret = 0)
[   26.308980] WiFi@C03L1,MtCmdSetMacTxRx() 10214: (ret = 0)
[   26.314361] WiFi@C00L1,ApAutoChannelAtBootUp() 336: PhyMode: 78, ACSChStat:0
[   26.321397] WiFi@C00L1,ApAutoChannelAtBootUp() 446: not A band
[   26.327211] WiFi@C08L1,ap_run_at_boot() 559: [41m ACS is disable !![m
[   26.333986] WiFi@C08L2,APStartUpForMbss() 909: ===>(caller:ap_inf_open+0x198/0x590 [mt_wifi]), mbss_idx:0, CfgMode:0
[   26.344492] WiFi@C15L2,APPMFInit() 133: [PMF] apidx=0, MFPC=0, MFPR=0, SHA256=0
[   26.352016] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
[   26.359519] WiFi@C03L1,MtCmdSetMacTxRx() 10214: (ret = 0)
[   26.364992] WiFi@C03L1,MtCmdSetMacTxRx() 10214: (ret = 0)
[   26.370470] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   26.379607] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=3, ctrl_ch2=0, cent_ch=3 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(1)
[   26.738396] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   26.747516] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=4, ctrl_ch2=0, cent_ch=4 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(1)
[   27.098409] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   27.107539] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=5, ctrl_ch2=0, cent_ch=5 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(1)
[   27.458370] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   27.467492] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=6, ctrl_ch2=0, cent_ch=6 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(1)
[   27.828358] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   27.837480] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=7, ctrl_ch2=0, cent_ch=7 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(1)
[   28.188388] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   28.197510] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=8, ctrl_ch2=0, cent_ch=8 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(1)
[   28.548374] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   28.557495] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=9, ctrl_ch2=0, cent_ch=9 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(1)
[   28.918369] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   28.927491] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=10, ctrl_ch2=0, cent_ch=10 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(1)
[   29.278401] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   29.287523] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=11, ctrl_ch2=0, cent_ch=11 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(1)
[   29.638358] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   29.647479] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=12, ctrl_ch2=0, cent_ch=12 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(1)
[   29.998373] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   30.007496] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=13, ctrl_ch2=0, cent_ch=13 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(1)
[   30.358384] WiFi@C00L1,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
[   30.367505] WiFi@C03L2,MtCmdChannelSwitch() 2530: ctrl_chl=6, ctrl_ch2=0, cent_ch=8 DBDCIdx=0, ChBand=0, BW=1, TXStream=4, RXStream=4, scan(0)
[   30.625309] WiFi@C08L2,ap_link_up() 7013: (caller:wdev_do_linkup+0x20/0x38 [mt_wifi]), wdev(0)
[   30.633915] WiFi@C01L2,wifi_sys_linkup() 1133: wdev idx = 0
[   30.639740] WiFi@C08L3,wifi_sys_op_lock() 791: wifi_sys_linkup+0x80/0x460 [mt_wifi](0): get wdev_op_lock.
[   30.649344] WiFi@C08L3,hw_ctrl_flow_v2_link_up() 72: wdev_idx=0
[   30.662030] WiFi@C08L3,wifi_sys_op_unlock() 806: release wdev_op_lock.
[   30.668587] WiFi@C00L1,TxCCKStreamCtrl() 21731: set wrong parameters
[   30.684725] PrintSrCmd:
[   30.684725] u1CmdSubId = 1, u1ArgNum = 0, u1DbdcIdx = 0, u1Status = 0
[   30.684725] u1DropTaIdx = 0, u1StaIdx = 0, u4Value = 0
[   30.698699] PrintSrCmd:
[   30.698699] u1CmdSubId = 5, u1ArgNum = 0, u1DbdcIdx = 0, u1Status = 0
[   30.698699] u1DropTaIdx = 0, u1StaIdx = 0, u4Value = 0
[   30.712675] PrintSrCmd:
[   30.712675] u1CmdSubId = 3, u1ArgNum = 0, u1DbdcIdx = 0, u1Status = 0
[   30.712675] u1DropTaIdx = 0, u1StaIdx = 0, u4Value = 1
[   30.726649] PrintSrCmd:
[   30.726649] u1CmdSubId = 23, u1ArgNum = 0, u1DbdcIdx = 0, u1Status = 0
[   30.726649] u1DropTaIdx = 0, u1StaIdx = 0, u4Value = 1
[   30.740727] PrintSrCmd:
[   30.740727] u1CmdSubId = 31, u1ArgNum = 0, u1DbdcIdx = 0, u1Status = 0
[   30.740727] u1DropTaIdx = 0, u1StaIdx = 0, u4Value = 0
[   30.754817] WiFi@C03L3,MtCmdTxPwrShowInfo() 13714: ucTxPowerInfoCatg: 2, BandIdx: 0 
[   30.762550] WiFi@C03L3,MtCmdTxPwrShowInfo() 13743: (ret = 0)
[   30.762643] [mtf_txpower_all_rate_info] band_idx:0 pwr:24 ChBand:2G ePAGain:0
[   30.768205] mtk_ppe_dev_register_hook : ineterface ra0 register (1)
[   30.782808] 7986@C12L1rax0,rt28xx_ap_ioctl() 860: interface is down, cmd [8be1] return!!!
wifid[wifi_exec_[   30.791055] 7986@C12L1rax0,rt28xx_ap_ioctl() 860: interface is down, cmd [8be1] return!!!
cmd:1957]: brctl[   30.800610] br-lan: port 6(ra0) entered blocking state
 addif br-lan ra[   30.807062] br-lan: port 6(ra0) entered disabled state
0
[   30.813848] device ra0 entered promiscuous mode
[   30.818496] br-lan: port 6(ra0) entered blocking state
[   30.823625] br-lan: port 6(ra0) entered forwarding state
[   30.829049] IPv6: ADDRCONF(NETDEV_CHANGE): br-lan: link becomes ready
<APSD_INFO>[path_selection_event_cb, 365] get ev[   30.839337] 7986@C01L2ra0,RTMP_COM_IoctlHandle() 1505: -> CMD_RTPRIV_IOCTL_VIRTUAL_INF_INIT
ent: 3

wifid[[   30.848055] 7986@C01L2ra0,RTMP_COM_IoctlHandle() 1482:  -> CMD_RTPRIV_IOCTL_VIRTUAL_INF_UP
wifi_exec_cmd:19[   30.857774] sta_inf_open(600)clear old scanTab here

57]: ifconfig ap[   30.864485] WiFi@C01L2,wifi_sys_open() 832: wdev idx = 4
cli0 up
<APSD_I[   30.871134] 7986@C08L3ra0,wifi_sys_op_lock() 791: wifi_sys_open+0x48/0x408 [mt_wifi](4): get wdev_op_lock.
NFO>[loop_avoida[   30.881875] 7986@C00L1ra0,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
nce_event_cb, 29[   30.892515] 7986@C03L2ra0,MtCmdChannelSwitch() 2530: ctrl_chl=6, ctrl_ch2=0, cent_ch=6 DBDCIdx=0, ChBand=0, BW=0, TXStream=4, RXStream=4, scan(0)
8] get netlink msg type: 6

<APSD_INFO>[loop_avoidance_configure_bridge_with_mode, 115] configure hyfi bridge with mode RE
<APSD_ERR>[apsd_get_wireless_link_state, 292] parse /tmp/wifi_runtime_info.2g failed
monitor doing 
<APSD_ERR>[apsd_get_wireless_link_state, 292] parse /tmp/wifi_runtime_info.5g failed
[   31.151938] WiFi@C00L2,dump_ht_cap() 29: Caller: wlan_operate_init+0xac/0x120 [mt_wifi]
[   31.159971] wdev_attr_update(): wdevId4 = 36:de:4b:4e:e0:6b
[   31.165565] 7986@C08L3ra0,hw_ctrl_flow_v2_open() 13: wdev_idx=4
[   31.171645] 7986@C08L3ra0,wifi_sys_op_unlock() 806: release wdev_op_lock.
[   31.178643] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
[   31.186036] mtk_ppe_dev_register_hook : ineterface apcli0 register (2)
wifid[wifi_exec_cmd:1957]: brctl[   31.195510] br-lan: port 7(apcli0) entered blocking state
 addif br-lan ap[   31.201553] br-lan: port 7(apcli0) entered disabled state
cli0
[   31.208494] device apcli0 entered promiscuous mode
[   31.213692] br-lan: port 7(apcli0) entered blocking state
[   31.219099] br-lan: port 7(apcli0) entered forwarding state
wifid[wifi_exec_[   31.225845] ath_band_steering_netlink_receive 161 pid = 2512

[   31.229806] 7986@C08L2rax0,mbss_virtual_if_open() 134:  ===> rax0
cmd:1957]: ifcon[   31.238507] 7986@C01L2rax0,RTMP_COM_IoctlHandle() 1505: -> CMD_RTPRIV_IOCTL_VIRTUAL_INF_INIT
fig rax0 up
[   31.248268] 7986@C01L2rax0,RTMP_COM_IoctlHandle() 1482:  -> CMD_RTPRIV_IOCTL_VIRTUAL_INF_UP
<APSD_INFO>[path[   31.257759] WiFi@C01L2,wifi_sys_open() 832: wdev idx = 2
_selection_event[   31.264804] 7986@C08L3rax0,wifi_sys_op_lock() 791: wifi_sys_open+0x48/0x408 [mt_wifi](2): get wdev_op_lock.
_cb, 365] get ev[   31.275599] [Nop_List_Backup] NopList Channel count:0 <----
ent: 3

{"sun"[   31.282621] [RadarStateCheck] Zero Wait State: 0
:[],"wed":[],"sa[   31.288466] 7986@C23L2rax0,RadarStateCheck() 265: [1;33m RD_SILENCE_MODE [m 
t":[],"tue":[],"[   31.297027] 7986@C23L2rax0,RadarStateCheck() 274: [DFS_ZDFS_ENHANCE], Pre CH: 0,	Pre BW: 0,	Pre CAC done: 0
thu":[],"fri":[][   31.308173] 7986@C23L2rax0,DfsCacNormalStart() 3462: [RDM] CAC 65 seconds start . Disable MAC TX
,"mon":[]}	
<AP[   31.318309] [41m[MY_DBG][m DfsCacNormalStart() pDot11h->RDMode=2
SD_INFO>[loop_av[   31.325931] 7986@C00L1rax0,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
oidance_event_cb[   31.336734] 7986@C03L2rax0,MtCmdChannelSwitch() 2530: ctrl_chl=36, ctrl_ch2=0, cent_ch=50 DBDCIdx=1, ChBand=1, BW=3, TXStream=4, RXStream=4, scan(0)
, 298] get netli[   31.351398] Per-port-per-queue mode is going to be enabled !
nk msg type: 6

/usr/sbin/tsched_conf -a power_schedule power_schedule '{"sun":[],"wed":[],"sat":[],"tue":[],"thu":[],"fri":[],"mon":[]}'	
<APSD_INFO>[loop_avoidance_configure_bridge_with_mode, 115] configure hyfi bridge with mode RE
<APSD_ERR>[apsd_get_wireless_link_[   31.381186] [41m[MY_DBG][m DfsCacNormalStart() pDot11h->RDMode=2
state, 292] pars[   31.388500] WiFi@C00L2,dump_ht_cap() 29: Caller: wlan_operate_init+0xac/0x120 [mt_wifi]
e /tmp/wifi_runt[   31.390582] S99sync (2563): drop_caches: 3
ime_info.2g fail[   31.397442] wdev_attr_update(): wdevId2 = 30:de:4b:4e:e0:6a
ed
<APSD_ERR>[a[   31.397506] WiFi@C12L1,wifi_sys_open() 874: wifi_sys_open():band1 group_idx[2]=4, bw_grp=16 omac=0
psd_get_wireless[   31.408304] BcnCheck start after 4300 ms (ra0)
[   31.409881] 7986@C08L3rax0,hw_ctrl_flow_v2_open() 13: wdev_idx=2
_link_state, 292[   31.433145] 7986@C08L3rax0,wifi_sys_op_unlock() 806: release wdev_op_lock.
] parse /tmp/wif[   31.440863] 7986@C02L2rax0,AsicRadioOnOffCtrl() 2279: DbdcIdx=1 RadioOn
i_runtime_info.5[   31.448700] mtk_band_steering_netlink_init 247 band steering netlink init ok

g failed
/usr/s[   31.456709] 7986@C00L2rax0,ApAutoChannelAtBootUp() 301: ----------------->
bin/tsched_conf [   31.464960] 7986@C00L0rax0,ApAutoChannelAtBootUp() 317: AutoChannelBootup[1] = 0
-u power_schedul[   31.473790] 7986@C03L1rax0,MtCmdSetMacTxRx() 10214: (ret = 0)
e	
<APSD_INFO>[[   31.480960] 7986@C03L1rax0,MtCmdSetMacTxRx() 10214: (ret = 0)
loop_avoidance_p[   31.487924] 7986@C00L1rax0,ApAutoChannelAtBootUp() 336: PhyMode: 177, ACSChStat:0
ick_bcast_nonrel[   31.496759] 7986@C00L1rax0,ApAutoChannelAtBootUp() 440: zero-wait DFS is not enabled
ay_port, 53] Pic[   31.505853] 7986@C08L1rax0,ap_run_at_boot() 559: [41m ACS is disable !![m
k apcli0 as broa[   31.514362] 7986@C08L2rax0,APStartUpForMbss() 909: ===>(caller:ap_inf_open+0x198/0x590 [mt_wifi]), mbss_idx:2, CfgMode:0
dcast nonrelay p[   31.526446] 7986@C15L2rax0,APPMFInit() 133: [PMF] apidx=2, MFPC=0, MFPR=0, SHA256=0
ort!
qos_toggle[   31.526471] wdev2 Turn off Edcca, prim-channel 36 Ap count 2!!
 debug default 2[   31.535649] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]

procd: - init [   31.551513] WiFi@C08L2,ap_link_up() 7013: (caller:wdev_do_linkup+0x20/0x38 [mt_wifi]), wdev(2)
complete -
[   31.561342] 7986@C01L2rax0,wifi_sys_linkup() 1133: wdev idx = 2
[   31.568501] 7986@C08L3rax0,wifi_sys_op_lock() 791: wifi_sys_linkup+0x80/0x460 [mt_wifi](2): get wdev_op_lock.
[   31.578521] 7986@C08L3rax0,hw_ctrl_flow_v2_link_up() 72: wdev_idx=2
[   31.591529] 7986@C08L3rax0,wifi_sys_op_unlock() 806: release wdev_op_lock.
[   31.610915] PrintSrCmd:
[   31.610915] u1CmdSubId = 1, u1ArgNum = 0, u1DbdcIdx = 1, u1Status = 0
[   31.610915] u1DropTaIdx = 0, u1StaIdx = 0, u4Value = 0
[   31.624907] PrintSrCmd:
[   31.624907] u1CmdSubId = 5, u1ArgNum = 0, u1DbdcIdx = 1, u1Status = 0
[   31.624907] u1DropTaIdx = 0, u1StaIdx = 0, u4Value = 0
[   31.638912] PrintSrCmd:
[   31.638912] u1CmdSubId = 3, u1ArgNum = 0, u1DbdcIdx = 1, u1Status = 0
[   31.638912] u1DropTaIdx = 0, u1StaIdx = 0, u4Value = 1
[   31.652899] PrintSrCmd:
[   31.652899] u1CmdSubId = 23, u1ArgNum = 0, u1DbdcIdx = 1, u1Status = 0
[   31.652899] u1DropTaIdx = 0, u1StaIdx = 0, u4Value = 1
[   31.666979] PrintSrCmd:
[   31.666979] u1CmdSubId = 31, u1ArgNum = 0, u1DbdcIdx = 1, u1Status = 0
[   31.666979] u1DropTaIdx = 0, u1StaIdx = 0, u4Value = 0
[   31.681085] 7986@C03L3rax0,MtCmdTxPwrShowInfo() 13714: ucTxPowerInfoCatg: 2, BandIdx: 1 
[   31.689176] 7986@C03L3rax0,MtCmdTxPwrShowInfo() 13743: (ret = 0)
[   31.689277] [mtf_txpower_all_rate_info] band_idx:1 pwr:30 ChBand:5G ePAGain:0
[   31.695181] mtk_ppe_dev_register_hook : ineterface rax0 register (3)
wifid[wifi_exec_[   31.710951] ath_band_steering_netlink_receive 161 pid = 2512

cmd:1957]: ifcon[   31.712745] 7986@C01L2apcli0,RTMP_COM_IoctlHandle() 1505: -> CMD_RTPRIV_IOCTL_VIRTUAL_INF_INIT
fig apclix0 up
[   31.726795] 7986@C01L2apcli0,RTMP_COM_IoctlHandle() 1482:  -> CMD_RTPRIV_IOCTL_VIRTUAL_INF_UP
[   31.736662] sta_inf_open(600)clear old scanTab here

[   31.741983] WiFi@C01L2,wifi_sys_open() 832: wdev idx = 5
[   31.747571] 7986@C08L3apcli0,wifi_sys_op_lock() 791: wifi_sys_open+0x48/0x408 [mt_wifi](5): get wdev_op_lock.
[   31.757542] [RadarStateCheck] Zero Wait State: 0
[   31.762156] 7986@C23L2apcli0,RadarStateCheck() 265: [1;33m RD_SILENCE_MODE [m 
[   31.769538] 7986@C23L2apcli0,RadarStateCheck() 274: [DFS_ZDFS_ENHANCE], Pre CH: 0,	Pre BW: 0,	Pre CAC done: 0
[   31.779445] 7986@C23L2apcli0,DfsCacNormalStart() 3462: [RDM] CAC 65 seconds start . Disable MAC TX
[   31.788391] [41m[MY_DBG][m DfsCacNormalStart() pDot11h->RDMode=2
[   31.794551] [41m[MY_DBG][m DfsCacNormalStart() pDot11h->RDMode=2
[   31.800882] WiFi@C00L2,dump_ht_cap() 29: Caller: wlan_operate_init+0xac/0x120 [mt_wifi]
[   31.808875] wdev_attr_update(): wdevId5 = 3a:de:4b:4e:e0:6b
[   31.814448] 7986@C08L3apcli0,hw_ctrl_flow_v2_open() 13: wdev_idx=5
[   31.821841] 7986@C08L3apcli0,wifi_sys_op_unlock() 806: release wdev_op_lock.
[   31.829058] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
[   31.836451] mtk_ppe_dev_register_hook : ineterface apclix0 register (4)
[   31.843412] br-lan: port 7(apcli0) entered disabled state
wifid[wifi_exec_cmd:1957]: brctl addif br-lan apclix0
<APSD_INFO>[path_selection_event_cb, 365] get event: 3

<APSD_INFO>[loop_avoidance_event_cb, 298] get netlink msg type: 4

[   31.880661] br-lan: port 8(apclix0) entered blocking state
[   31.886136] br-lan: port 8(apclix0) entered disabled state
[   31.891805] device apclix0 entered promiscuous mode
wifid[wifi_exec_cmd:1957]: ifconfig rax0 down
<APSD_INFO>[path_[   31.902679] 7986@C08L2apcli0,mbss_virtual_if_close() 171:  ===> rax0
selection_event_[   31.909215] 7986@C01L2apcli0,RTMP_COM_IoctlHandle() 1493:  -> CMD_RTPRIV_IOCTL_VIRTUAL_INF_DOWN
cb, 365] get eve[   31.919460] WiFi@C08L2,ap_link_down() 7031: (caller:wdev_do_linkdown+0x20/0x38 [mt_wifi]), wdev(2)
nt: 3

<APSD_I[   31.929873] 7986@C08L1apcli0,UpdateBeaconHandler() 2003: wdev(2) bss not ready (state:2, caller:ap_link_down+0x94/0x128 [mt_wifi])!!
NFO>[loop_avoida[   31.942919] 7986@C01L2apcli0,wifi_sys_linkdown() 1257: wdev idx = 2
nce_event_cb, 29[   31.950785] 7986@C08L3apcli0,wifi_sys_op_lock() 791: wifi_sys_linkdown+0x58/0x378 [mt_wifi](2): get wdev_op_lock.
8] get netlink m[   31.962119] 7986@C08L3apcli0,hw_ctrl_flow_v2_link_down() 123: wdev_idx=2
sg type: 6

<APSD_INFO>[loop_avoidance_configu[   31.973171] 7986@C08L3apcli0,wifi_sys_op_unlock() 806: release wdev_op_lock.
re_bridge_with_m[   31.981370] WiFi@C01L2,wifi_sys_close() 912: wdev idx = 2
ode, 115] config[   31.988469] 7986@C08L3apcli0,wifi_sys_op_lock() 791: wifi_sys_close+0x88/0x270 [mt_wifi](2): get wdev_op_lock.
ure hyfi bridge [   31.999488] 7986@C08L3apcli0,hw_ctrl_flow_v2_close() 41: wdev_idx=2
with mode RE
<A[   32.007201] 7986@C08L3apcli0,wifi_sys_op_unlock() 806: release wdev_op_lock.
PSD_ERR>[apsd_ge[   32.015497] 7986@C01L2apcli0,RTMP_COM_IoctlHandle() 1516: -> CMD_RTPRIV_IOCTL_VIRTUAL_INF_DEINIT
t_wireless_link_[   32.025632] extif_put_dev(rax0)
state, 292] pars[   32.030133] mtk_ppe_dev_unregister_hook : ineterface rax0 set null (3)
e /tmp/wifi_runtime_info.2g fail[   32.039752] 7986@C12L1rax0,rt28xx_ap_ioctl() 860: interface is down, cmd [8be1] return!!!
ed
<APSD_ERR>[a[   32.048992] 7986@C12L1ra1,rt28xx_ap_ioctl() 860: interface is down, cmd [8be1] return!!!
psd_get_wireless[   32.058415] 7986@C12L1rax1,rt28xx_ap_ioctl() 860: interface is down, cmd [8be1] return!!!
_link_state, 292] parse /tmp/wifi_runtime_info.5g failed
<APSD_INFO>[loop_avoidance_pick_bcast_nonrelay_port, 53] Pick apclix0 as broadcast nonrelay port!
wifid[backend_ap_tpie_info_stop:5617]: call tp_ie_info_stop
wifid[tp_ie_info_stop:[   32.087268] Set_RTSThreshold_Proc: set wdev0 rts length threshold=2346(0x92a)
5073]: length = 0, flags = 37377[   32.098507] Set_FragThreshold_Proc::set wdev0 FragThreshold=2346)

wifid[tp_ie_info_stop:5081]: s[   32.107129] sync for : ra0
top tpie ret:0
wifid[tp_ie_info_stop:5073]: len[   32.113994] 7986@C12L3ra0,Set_DtimPeriod_Proc() 14223: (i/f 0) (DtimPeriod=1)
gth = 0, flags = 37377
wifid[tp[   32.124106] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
_ie_info_stop:5081]: stop tpie ret:-1
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set RTSThreshold=23[   32.139199] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
46
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set Fr[   32.149969] 7986@C12L1ra0,RTMPAPPrivIoctlSet() 3730: IOCTL::(iwpriv) Command not Support [EDCCACtrl=1]
agThreshold=2346
wifid[wifi_exec_cmd:1957]: iwp[   32.163422] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
riv ra0 set BeaconPeriod=100
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set DtimPeriod=1
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set [   32.182063] 7986@C08L3ra0,ap_mlme_deauth_req_action() 223:  ---> wcid = 32767
HtGi=1
wifid[wi[   32.190659] 7986@C15L2ra0,APPMFInit() 133: [PMF] apidx=0, MFPC=0, MFPR=0, SHA256=0
fi_exec_cmd:1957[   32.199433] 7986@C01L2ra0,wifi_sys_linkdown() 1257: wdev idx = 0
]: iwpriv ra0 se[   32.199614] 7986@C03L2ra0,ExtEventBeaconLostHandler() 169: FW EVENT (30:de:4b:4e:e0:6b), Reason 0x10
t BasicRate=15
[   32.207041] 7986@C08L3ra0,wifi_sys_op_lock() 791: wifi_sys_linkdown+0x58/0x378 [mt_wifi](0): get wdev_op_lock.
wifid[wifi_exec_[   32.217247] 7986@C03L1ra0,ExtEventBeaconLostHandler() 175:   AP Beacon OFF!!!
cmd:1957]: iwpri[   32.237133] 7986@C08L3ra0,hw_ctrl_flow_v2_link_down() 123: wdev_idx=0
v ra0 set HtBssCoex=0
wifid[wifi_exec_cmd:1957][   32.247875] 7986@C08L3ra0,wifi_sys_op_unlock() 806: release wdev_op_lock.
: iwpriv ra0 set[   32.255848] WiFi@C01L2,wifi_sys_close() 912: wdev idx = 0
 HtAmsdu=1
wifi[   32.262891] 7986@C08L3ra0,wifi_sys_op_lock() 791: wifi_sys_close+0x88/0x270 [mt_wifi](0): get wdev_op_lock.
d[wifi_exec_cmd:[   32.273749] 7986@C08L3ra0,hw_ctrl_flow_v2_close() 41: wdev_idx=0
1957]: iwpriv ra[   32.281219] 7986@C08L3ra0,wifi_sys_op_unlock() 806: release wdev_op_lock.
0 set EDCCACtrl=[   32.289243] WiFi@C01L2,wifi_sys_open() 832: wdev idx = 0
1
wifid[wifi_ex[   32.296148] 7986@C08L3ra0,wifi_sys_op_lock() 791: wifi_sys_open+0x48/0x408 [mt_wifi](0): get wdev_op_lock.
ec_cmd:1957]: iw[   32.307035] 7986@C00L1ra0,mt7986_apply_dpd_flatness_data() 619: eeprom 0x19a bit 0 is 0, do runtime cal
priv ra0 set Wmm[   32.317674] 7986@C03L2ra0,MtCmdChannelSwitch() 2530: ctrl_chl=6, ctrl_ch2=0, cent_ch=8 DBDCIdx=0, ChBand=0, BW=1, TXStream=4, RXStream=4, scan(0)
Capable=1
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set NoForwarding=0
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set NoForwardingBTNBSSID=0
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set UAPSDCapable=1
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set SSID=\T\P\-\L\i\n\k\_\E\x\t\e\n\d\e\r
[   32.581683] WiFi@C00L2,dump_ht_cap() 29: Caller: wlan_operate_init+0xac/0x120 [mt_wifi]
[   32.589691] wdev_attr_update(): wdevId0 = 30:de:4b:4e:e0:6b
[   32.595259] WiFi@C12L1,wifi_sys_open() 874: wifi_sys_open():band0 group_idx[0]=0, bw_grp=16 omac=0
[   32.604217] 7986@C08L3ra0,hw_ctrl_flow_v2_open() 13: wdev_idx=0
[   32.610286] 7986@C08L3ra0,wifi_sys_op_unlock() 806: release wdev_op_lock.
[   32.617083] 7986@C01L2ra0,wifi_sys_linkup() 1133: wdev idx = 0
[   32.623196] 7986@C08L3ra0,wifi_sys_op_lock() 791: wifi_sys_linkup+0x80/0x460 [mt_wifi](0): get wdev_op_lock.
[   32.633127] 7986@C08L3ra0,hw_ctrl_flow_v2_link_up() 72: wdev_idx=0
[   32.646059] 7986@C08L2ra0,bcn_buf_init() 170: BcnPkt is allocated!, bcn offload=1
[   32.653566] 7986@C08L3ra0,wifi_sys_op_unlock() 806: release wdev_op_lock.
[   32.660366] 7986@C03L3ra0,bssUpdateHighPriRateEAPOL() 4043: bssUpdateHighPriRateEAPOL (BSS_INFO_HIGHPRI_INFO), CmdBssInfoHighPriRate.u2HighPriTransmit = 0
[   32.674325] 7986@C03L3ra0,bssUpdateHighPriRateEAPOL() 4043: bssUpdateHighPriRateEAPOL (BSS_INFO_HIGHPRI_INFO), CmdBssInfoHighPriRate.u2HighPriTransmit = 0
[   32.688248] Set_HighPriMcs(): input param error
[   32.688248] Usage: iwpriv [inf] set HighPriMcs=[frame_type]-[mcs]
[   32.688248] The frame type format is 0 for ARP, 1 for DHCP, 2 for EAPOL
[   32.688248] MCS must in range of 0 ~ 3 and 8 ~ 11 for CCK Mode
[   32.688248] MCS must in range from 0 to 7 for OFDM Mode
[   32.716473] 7986@C12L1rax0,rt28xx_ap_ioctl() 860: interface is down, cmd [8be1] return!!!
wifid[wifi_exec_cmd:1957]: iwpri[   32.726998] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
v rax0 set RTSThreshold=2346
wi[   32.738059] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
fid[wifi_exec_cmd:1957]: iwpriv [   32.749008] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
rax0 set FragThreshold=2346
wif[   32.759950] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
id[wifi_exec_cmd:1957]: iwpriv r[   32.770840] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
ax0 set BeaconPeriod=100
wifid[wifi_exec_cmd:19[   32.782059] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
57]: iwpriv rax0 set DtimPeriod=[   32.794115] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
1
wifid[wifi_exec_cmd:1957]: iw[   32.804995] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
priv rax0 set HtGi=1
wifid[wifi[   32.815940] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
_exec_cmd:1957]: iwpriv rax0 set[   32.826886] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
 BasicRate=15
wifid[wifi_exec_c[   32.837791] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
md:1957]: iwpriv rax0 set HtBssC[   32.848798] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
oex=0
wifid[wifi_exec_cmd:1957][   32.859714] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
: iwpriv rax0 set HtAmsdu=1
wif[   32.870603] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
id[wifi_exec_cmd:1957]: iwpriv r[   32.881569] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
ax0 set EDCCACtrl=1
wifid[wifi_[   32.892469] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
exec_cmd:1957]: iwpriv rax0 set [   32.903422] SetApCliEnableByWdev: I/F(apcli1) (enable = 0)
WmmCapable=1
wifid[wifi_exec_cmd:1957]: iwpriv [   32.911843] SetApCliEnableByWdev: I/F(apcli1) (enable = 0)
rax0 set NoForwarding=0
wifid[wifi_exec_cmd:1957]: iwpriv rax0 set NoForwardingBTNBSSID=0
wifid[wifi_exec_cmd:1957]: iwpriv ra[   32.928913] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
x0 set UAPSDCapable=1
wifid[wif[   32.940563] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
i_exec_cmd:1957]: iwpriv rax0 se[   32.951424] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
t SSID=\T\P\-\L\i\n\k\_\E\x\t\e\[   32.962456] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
n\d\e\r\_\5\G\H\z
wifid[read_wp[   32.973240] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
s_result_flag:2657]: /tmp/mtk_wp[   32.984323] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
sdflag is not exit
wifid[wifi_e[   32.995184] 7986@C12L1rax0,rt28xx_ap_ioctl() 893: interface is down, cmd [8beb] return!!!
xec_cmd:1957]: iwpriv rax0 set C[   33.006171] 
[   33.006171] I/F(apcli1) Set_ApCli_Mesh_Rule_Enable_Proc::(enable=0)
ountryCode=DE
wifid[wifi_exec_c[   33.016651] 
[   33.016651] I/F(apcli0) Set_ApCli_Conn_Specific_NodeId_Proc::(enable=0,nodeId=0000)
md:1957]: iwpriv rax0 set Countr[   33.028520] 
[   33.028520] I/F(apcli1) Set_ApCli_Conn_Specific_NodeId_Proc::(enable=0,nodeId=0000)
yRegionABand=1
wifid[wifi_exec_[   33.040448] 7986@C12L1apcli0,RTMPAPPrivIoctlSet() 3730: IOCTL::(iwpriv) Command not Support [apgroupid=00-00-00-00-00-00-00-00]
cmd:1957]: iwpriv apclix0 set Ap[   33.054661] 7986@C12L1apclix0,RTMPAPPrivIoctlSet() 3730: IOCTL::(iwpriv) Command not Support [apgroupid=00-00-00-00-00-00-00-00]
CliEnable=0
wifid[wifi_exec_cmd:1957]: iwpriv apclix0 set ApCli[   33.071362] SetApCliEnableByWdev: I/F(apcli1) (enable = 1)
Enable=0
wifid[_setSTA_security_settings:2099]: ApCli none
wif[   33.082413] [Nop_List_Backup] NopList Channel count:0 <----
id[wifi_exec_cmd[   33.088555] 7986@C23L1ra0,DfsBuildChannelList() 3551: [RDM]: wdev is not 5G.
:1957]: iwpriv apclix0 set ApCli[   33.099380] SetApCliEnableByWdev: I/F(apcli0) (enable = 0)
AuthMode=OPEN
wifid[wifi_exec_c[   33.107614] SetApCliEnableByWdev: I/F(apcli0) (enable = 0)
md:1957]: iwpriv apclix0 set ApCliEncrypType=NONE
wifid[wifi_exec_cmd:1957]: iwpriv apclix0 set ApCliSsid=
wifid[wifi_exec_cmd[   33.123419] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
:1957]: iwpriv r[   33.131881] WiFi@C02L1,hc_radio_res_request() 1420: wdev=1, hobj is not ready!
ax0 set HtBw=1
[   33.140617] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
wifid[wifi_exec_[   33.149331] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
cmd:1957]: iwpriv rax0 set VhtBw=2
wifid[wifi_e[   33.160802] WiFi@C00L2,dump_ht_cap() 29: Caller: wlan_operate_init+0xac/0x120 [mt_wifi]
xec_cmd:1957]: i[   33.170138] [Nop_List_Backup] NopList Channel count:0 <----
wpriv rax0 set W[   33.177067] 7986@C23L1ra0,DfsBuildChannelList() 3551: [RDM]: wdev is not 5G.
irelessMode=17
[   33.185620] WiFi@C00L2,dump_ht_cap() 29: Caller: dump_ht_cap+0x10/0x20 [mt_wifi]
wifid[wifi_exec_cmd:1957]: iwpriv rax0 set twtsupport=1:0
wifid[wifi_exec_cmd:1957]: iwpriv rax0 set MuOfdma=0
wifid[wifi_exec_cmd:1957]: iwpriv rax0 set MuMimo=0
wifid[wifi_exec_cmd:1957]: iwpriv rax0 set TxPower=100
wifid[wifi_exec_cmd:1957]: iwpriv apclix0 set ApCl[   33.216989] 7986@C08L3ra0,ap_mlme_deauth_req_action() 223:  ---> wcid = 32767
iMeshRule=0
wif[   33.225009] 7986@C15L2ra0,APPMFInit() 133: [PMF] apidx=0, MFPC=0, MFPR=0, SHA256=0
id[wifi_exec_cmd[   33.233855] 7986@C01L2ra0,wifi_sys_linkdown() 1257: wdev idx = 0
:1957]: iwpriv a[   33.241333] 7986@C03L2ra0,ExtEventBeaconLostHandler() 169: FW EVENT (30:de:4b:4e:e0:6b), Reason 0x10
pcli0 set ApCliC[   33.241490] 7986@C08L3ra0,wifi_sys_op_lock() 791: wifi_sys_linkdown+0x58/0x378 [mt_wifi](0): get wdev_op_lock.
onnSpecificNodeI[   33.251687] 7986@C03L1ra0,ExtEventBeaconLostHandler() 175:   AP Beacon OFF!!!
d=0:0
wifid[wif[   33.271587] 7986@C08L3ra0,hw_ctrl_flow_v2_link_down() 123: wdev_idx=0
i_exec_cmd:1957]: iwpriv apclix0 set ApCliConnSp[   33.282316] 7986@C08L3ra0,wifi_sys_op_unlock() 806: release wdev_op_lock.
ecificNodeId=0:0[   33.290291] WiFi@C01L2,wifi_sys_close() 912: wdev idx = 0

wifid[wifi_exe[   33.297314] 7986@C08L3ra0,wifi_sys_op_lock() 791: wifi_sys_close+0x88/0x270 [mt_wifi](0): get wdev_op_lock.
c_cmd:1957]: iwp[   33.308212] 7986@C08L3ra0,hw_ctrl_flow_v2_close() 41: wdev_idx=0
riv apcli0 set a[   33.315788] 7986@C08L3ra0,wifi_sys_op_unlock() 806: release wdev_op_lock.
pgroupid=00-00-0[   33.323685] WiFi@C01L2,wifi_sys_open() 832: wdev idx = 0
0-00-00-00-00-00[   33.330627] 7986@C08L3ra0,wifi_sys_op_lock() 791: wifi_sys_open+0x48/0x408 [mt_wifi](0): get wdev_op_lock.

wifid[wifi_exe[   33.341560] WiFi@C00L2,dump_ht_cap() 29: Caller: wlan_operate_init+0xac/0x120 [mt_wifi]
c_cmd:1957]: iwp[   33.350726] wdev_attr_update(): wdevId0 = 30:de:4b:4e:e0:6b
riv apclix0 set [   33.357669] WiFi@C12L1,wifi_sys_open() 874: wifi_sys_open():band0 group_idx[0]=0, bw_grp=16 omac=0
apgroupid=00-00-[   33.367991] 7986@C08L3ra0,hw_ctrl_flow_v2_open() 13: wdev_idx=0
00-00-00-00-00-0[   33.375405] 7986@C08L3ra0,wifi_sys_op_unlock() 806: release wdev_op_lock.
0
wifid[wifi_ex[   33.383421] 7986@C01L2ra0,wifi_sys_linkup() 1133: wdev idx = 0
ec_cmd:1957]: iw[   33.390881] 7986@C08L3ra0,wifi_sys_op_lock() 791: wifi_sys_linkup+0x80/0x460 [mt_wifi](0): get wdev_op_lock.
priv apclix0 set[   33.401837] 7986@C08L3ra0,hw_ctrl_flow_v2_link_up() 72: wdev_idx=0
 skip_cac=1
wifid[wifi_exec_cmd:1957]: iwpriv apclix0 set ApCliEnable=1
wifid[[   33.416119] 7986@C08L2ra0,bcn_buf_init() 170: BcnPkt is allocated!, bcn offload=1
_init_band_wifi:[   33.423801] 7986@C08L3ra0,wifi_sys_op_unlock() 806: release wdev_op_lock.
2348]: tmp_wps_f[   33.431957] 7986@C03L3ra0,bssUpdateHighPriRateEAPOL() 4043: bssUpdateHighPriRateEAPOL (BSS_INFO_HIGHPRI_INFO), CmdBssInfoHighPriRate.u2HighPriTransmit = 0
lag=00,band = 1
[   33.447165] 7986@C03L3ra0,bssUpdateHighPriRateEAPOL() 4043: bssUpdateHighPriRateEAPOL (BSS_INFO_HIGHPRI_INFO), CmdBssInfoHighPriRate.u2HighPriTransmit = 0

wifid[wifi_exec[   33.462334] Set_HighPriMcs(): input param error
[   33.462334] Usage: iwpriv [inf] set HighPriMcs=[frame_type]-[mcs]
[   33.462334] The frame type format is 0 for ARP, 1 for DHCP, 2 for EAPOL
[   33.462334] MCS must in range of 0 ~ 3 and 8 ~ 11 for CCK Mode
[   33.462334] MCS must in range from 0 to 7 for OFDM Mode
_cmd:1957]: iwpriv ra0 set Count[   33.494420] 
[   33.494420] I/F(apcli0) Set_ApCli_Mesh_Rule_Enable_Proc::(enable=0)
ryCode=DE
wifid[wifi_exec_cmd:1[   33.504841] 
[   33.504841] I/F(apcli0) Set_ApCli_Conn_Specific_NodeId_Proc::(enable=0,nodeId=0000)
957]: iwpriv ra0 set CountryRegi[   33.516614] 
[   33.516614] I/F(apcli1) Set_ApCli_Conn_Specific_NodeId_Proc::(enable=0,nodeId=0000)
on=1
wifid[wifi_exec_cmd:1957]:[   33.528570] 7986@C12L1apcli0,RTMPAPPrivIoctlSet() 3730: IOCTL::(iwpriv) Command not Support [apgroupid=00-00-00-00-00-00-00-00]
 iwpriv apcli0 set ApCliEnable=0[   33.542788] 7986@C12L1apclix0,RTMPAPPrivIoctlSet() 3730: IOCTL::(iwpriv) Command not Support [apgroupid=00-00-00-00-00-00-00-00]

wifid[wifi_exec_cmd:1957]: iwpriv apcli0 set ApCliEnable=0
wi[   33.559516] SetApCliEnableByWdev: I/F(apcli0) (enable = 1)
fid[_setSTA_security_settings:2099]: ApCli none
wifid[wifi_exec_cmd:1957]: iwpriv apcli0 set ApCliAuthMode=OPEN
wifid[wifi_exec_cmd:1957]: iwpriv apcli0 set ApCliEncrypType=NONE
wifid[wifi_exec_cmd:1957]: iwpriv apcli0 set ApCliSsid=
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set HtBw=1
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set WirelessMode=16
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set twtsupport=1:0
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set MuOfdma=0
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set MuMimo=0
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set TxPower=100
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set AuthMode=OPEN
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set EncrypType=NONE
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set HideSSID=0
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set UAPSDCapable=1
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set SSID=\T\P\-\L[   33.638760] 7986@C09L2apcli0,cntl_connect_request() 391: type=2,len=0,caller:ApCliIfUp+0x1a0/0x370 [mt_wifi]
\i\n\k\_\E\x\t\e[   33.649046] 7986@C09L2apcli0,cntl_connect_request() 391: type=2,len=0,caller:ApCliIfUp+0x1a0/0x370 [mt_wifi]
\n\d\e\r
wifid[wifi_exec_cmd:1957]: iwpriv apcli0 set ApCliMeshRule=0
wifid[wifi_exec_cmd:1957]: iwpriv apcli0 set ApCliConnSpecificNodeId=0:0
wifid[wifi_exec_cmd:1957]: iwpriv apclix0 set ApCliConnSpecificNodeId=0:0
wifid[wifi_exec_cmd:1957]: iwpriv apcli0 set apgroupid=00-00-00-00-00-00-00-00
wifid[wifi_exec_cmd:1957]: iwpriv apclix0 set apgroupid=00-00-00-00-00-00-00-00
wifid[wifi_exec_cmd:1957]: iwpriv apcli0 set skip_cac=1
wifid[wifi_exec_cmd:1957]: iwpriv apcli0 set ApCliEnable=1
wifid[_init_band_wifi:2348]: tmp_wps_flag=00,band = 0
wifid[wifi_exec_cmd:1957]: echo 1 > /tmp/wifi_mod_exist
wifid[wifi_exec_cmd:1957]: echo 'ra0:rax0:eth1:lan1:lan2:ra1:rax1' > /proc/dhcp_hook/dhcp_hook_dev_name
wifid[wifi_exec_cmd:1957]: /etc/init.d/wps start
table: 0x37ffa320	
on	
off	
wpsd[main:2069]: wpsd has been started
wifid[_a[   33.862656] 7986@C12L2ra0,Set_ACLClearAll_Proc() 15567: The Access Control List is empty!
cl_reload:743]: [   33.871771] 7986@C12L2ra0,Set_ACLClearAll_Proc() 15568: No need to clear the Access Control List!
===>> system mode: 0, mesh_enable: 0 


wifid[wifi_exec_cmd:1957]: iwpriv ra0 set ACLClearAll=1
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set AccessPolicy=1
wifid[wifi_exec_cmd:1957]: wifi chaninfo DE
wifid[wifi_exec_cmd:1957]: /etc/init.d/access_control reload
[   34.188617] 7986@C12L1rax0,rt28xx_ap_ioctl() 860: interface is down, cmd [8be1] return!!!
[   34.208294] BcnCheck start after 1800 ms (ra0)
@@@@@@@@ap follow sta ...
wpsd[wps_ubus_wifi_cb:125]: Message: {"action":8}
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
[   36.188613] 7986@C12L1rax0,rt28xx_ap_ioctl() 860: interface is down, cmd [8be1] return!!!
wifid[wifi_exec_cmd:1957]: wifi setscantime 15 18
wifid[wifi_exec_cmd:1957]: wifi setprescantime 15 18
wifid[wifi_exec_cmd:1957]: wifi prescanstat 1
wifid[wifi_exec_cmd:1957]: wifi doprescan 3
wpsd[wps_ubus_wifi_cb:125]: Message: {"action":4}
wifid[_start_prescan:1241]: _start_prescan!

wifid[wifi_exec_cmd:1957]: /sbin/smp.sh wifi
wifid[mtk_init_platform:7559]: Wifi start begin time: 21, end time:37 


wifid[_mark_down_reboot_time_to_use:6119]: Wifi start need time: 16


wifid[_mark_down_reboot_time_to_use:6124]: Mark down system start need time: 37


wifid[mtk_init_platform:7561]: mtk init platform end.
wifid[wifi_exec_cmd:1957]: echo 1 > /tmp/wifi_init_finish_flag
wifid[_acl_reload:743]: ===>> system mode: 0, mesh_enable: 0 


wifid[wifi_exec_cmd:1957]: iwpriv ra0 set AccessPolicy=0
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set PartialScanNumOfCh=2
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set PartialScan=2
wifid[wifi_exec_cmd:1957]: iwpriv apclix0 set SiteSurvey=
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
wifid[wifi_exec_cmd:1957]: wifi setprescantime 12 15
wifid[wifi_exec_cmd:1957]: wifi setscantime 12 15
wifid[wifi_exec_cmd:1957]: wifi setprescantime 9 12
wifid[wifi_exec_cmd:1957]: wifi setscantime 9 12
wifid[wifi_exec_cmd:1957]: wifi setprescantime 6 9
wifid[wifi_exec_cmd:1957]: wifi setscantime 6 9
wifid[wifi_exec_cmd:1957]: wifi setprescantime 3 6
wifid[wifi_exec_cmd:1957]: wifi setscantime 3 6
wifid[wifi_exec_cmd:1957]: wifi setprescantime 0 3
wifid[wifi_exec_cmd:1957]: wifi setscantime 0 3
wifid[wifi_exec_cmd:1957]: wifi getprescanresult 0
wifid[_scan_result:468]: ============band: 0 ============Scan is not running: Ch  SSID                             LENBSSID               Security               Siganl(%)W-Mode      ExtCH  NT WPS DPID TPIE                                                                                                                           !!!!!

wpsd[wps_ubus_wifi_cb:125]: Message: {"action":5,"band":0}
wifid[wifi_exec_cmd:1957]: wifi setscantime 15 3
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
wifid[wifi_exec_cmd:1957]: wifi setprescantime 0 0
wifid[wifi_exec_cmd:1957]: wifi setscantime 0 0
wifid[wifi_exec_cmd:1957]: wifi getprescanresult 1
wifid[_scan_result:468]: ============band: 1 ============Scan is not running: Ch  SSID                             LENBSSID               Security               Siganl(%)W-Mode      ExtCH  NT WPS DPID TPIE                                                                                                                           !!!!!

wpsd[wps_ubus_wifi_cb:125]: Message: {"action":5,"band":1}
wifid[wifi_exec_cmd:1957]: wifi setscantime 15 18
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
wifid[wifi_exec_cmd:1957]: wifi doprescan 3
wpsd[wps_ubus_wifi_cb:125]: Message: {"action":4}
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set PartialScanNumOfCh=2
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set PartialScan=2
wifid[wifi_exec_cmd:1957]: iwpriv apclix0 set SiteSurvey=
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
wifid[wifi_exec_cmd:1957]: wifi setscantime 12 15
wifid[wifi_exec_cmd:1957]: wifi setscantime 9 12
wifid[wifi_exec_cmd:1957]: wifi setscantime 6 9
wifid[wifi_exec_cmd:1957]: wifi setscantime 3 6
wifid[wifi_exec_cmd:1957]: wifi setscantime 0 3
wifid[wifi_exec_cmd:1957]: wifi getprescanresult 0
wifid[_scan_result:468]: ============band: 0 ============Scan is not running: Ch  SSID                             LENBSSID               Security               Siganl(%)W-Mode      ExtCH  NT WPS DPID TPIE                                                                                                                           !!!!!

wpsd[wps_ubus_wifi_cb:125]: Message: {"action":5,"band":0}
wifid[wifi_exec_cmd:1957]: wifi setscantime 15 3
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
wifid[wifi_exec_cmd:1957]: wifi setscantime 0 0
wifid[wifi_exec_cmd:1957]: wifi getprescanresult 1
wifid[_scan_result:468]: ============band: 1 ============Scan is not running: Ch  SSID                             LENBSSID               Security               Siganl(%)W-Mode      ExtCH  NT WPS DPID TPIE                                                                                                                           !!!!!

wpsd[wps_ubus_wifi_cb:125]: Message: {"action":5,"band":1}
wifid[wifi_exec_cmd:1957]: wifi setscantime 15 18
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
wifid[wifi_exec_cmd:1957]: wifi doprescan 3
wpsd[wps_ubus_wifi_cb:125]: Message: {"action":4}
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set PartialScanNumOfCh=2
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set PartialScan=2
wifid[wifi_exec_cmd:1957]: iwpriv apclix0 set SiteSurvey=
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
wifid[wifi_exec_cmd:1957]: wifi setscantime 12 15
wifid[wifi_exec_cmd:1957]: wifi setscantime 9 12
wifid[wifi_exec_cmd:1957]: wifi setscantime 6 9
wifid[wifi_exec_cmd:1957]: wifi setscantime 3 6
wifid[wifi_exec_cmd:1957]: wifi setscantime 0 3
wifid[wifi_exec_cmd:1957]: wifi getprescanresult 0
wifid[_scan_result:468]: ============band: 0 ============Scan is not running: Ch  SSID                             LENBSSID               Security               Siganl(%)W-Mode      ExtCH  NT WPS DPID TPIE                                                                                                                           !!!!!

wpsd[wps_ubus_wifi_cb:125]: Message: {"action":5,"band":0}
wifid[wifi_exec_cmd:1957]: wifi setscantime 15 3
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
wifid[wifi_exec_cmd:1957]: wifi setscantime 0 0
wifid[wifi_exec_cmd:1957]: wifi getprescanresult 1
wifid[_scan_result:468]: ============band: 1 ============Scan is not running: Ch  SSID                             LENBSSID               Security               Siganl(%)W-Mode      ExtCH  NT WPS DPID TPIE                                                                                                                           !!!!!

wpsd[wps_ubus_wifi_cb:125]: Message: {"action":5,"band":1}
wifid[wifi_exec_cmd:1957]: wifi setscantime 15 18
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
wifid[wifi_exec_cmd:1957]: wifi doprescan 3
wpsd[wps_ubus_wifi_cb:125]: Message: {"action":4}
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set PartialScanNumOfCh=2
wifid[wifi_exec_cmd:1957]: iwpriv ra0 set PartialScan=2
wifid[wifi_exec_cmd:1957]: iwpriv apclix0 set SiteSurvey=
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
wifid[wifi_exec_cmd:1957]: wifi setscantime 12 15
wifid[wifi_exec_cmd:1957]: wifi setscantime 9 12
wifid[wifi_exec_cmd:1957]: wifi setscantime 6 9
wifid[wifi_exec_cmd:1957]: wifi setscantime 3 6
wifid[wifi_exec_cmd:1957]: wifi setscantime 0 3
wifid[wifi_exec_cmd:1957]: wifi getprescanresult 0
wifid[_scan_result:468]: ============band: 0 ============Scan is not running: Ch  SSID                             LENBSSID               Security               Siganl(%)W-Mode      ExtCH  NT WPS DPID TPIE                                                                                                                           !!!!!

wpsd[wps_ubus_wifi_cb:125]: Message: {"action":5,"band":0}
wifid[wifi_exec_cmd:1957]: wifi setscantime 15 3
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
wifid[wifi_exec_cmd:1957]: wifi setscantime 0 0
wifid[wifi_exec_cmd:1957]: wifi getprescanresult 1
wifid[_scan_result:468]: ============band: 1 ============Scan is not running: Ch  SSID                             LENBSSID               Security               Siganl(%)W-Mode      ExtCH  NT WPS DPID TPIE                                                                                                                           !!!!!

wpsd[wps_ubus_wifi_cb:125]: Message: {"action":5,"band":1}
wifid[wifi_exec_cmd:1957]: wifi setscantime 15 18
wpsd[mtk_process_ibus_msg:997]: ibus event: 3
```